\section{Introduction}

--- STILL DRAFTING ---

Computational analysis has become an integral part of modern-day plant pathology. High-quality genome sequences are being generated for multiple plant and pathogen species including banana and \ac{Fo}. The \ac{ncbi} genome database currently holds 17 genome entries for the Musaceae family and 736 genome entries for \ac{Fo}; 17 of which are from \ac{Focub}, including a high-quality draft genome for \ac{Focub4} \parencite{Warmington2019} and \ac{Focub1} \parencite{Asai2019} (\href{https://www.ncbi.nlm.nih.gov/data-hub/genome/}{https://www.ncbi.nlm.nih.gov/data-hub/genome/} Accessed: 17/01/2024). Many bioinformatic tools have been developed to analyse the increasing number of genome sequences. For instance, \textcite{Park2010} highlight integrated platforms supporting strain identification, phylogenetics, comparative genomics and knowledge sharing for \textit{Fusarium}. The use of genomics to identify candidate effectors in \ac{Fo} is becoming commonplace and is contributing to our understanding of \ac{Fo} classification, evolution, and cryptic host specificity. 

Generally, fungal effectors do not share sequence homology or conserved motifs, so computational prediction of fungal candidate effectors is challenging \parencite{Sperschneider2022, Todd2022}. Broadly, fungal effectors have been predicted using a set of common characteristics; the presence of a secretion signal, sequence length $\leq$ 300 amino acids, and cysteine richness \parencite{Sperschneider2015}. Other features sometimes used to filter protein sets include the lack of a transmembrane domain; increased expression during host interaction; restricted taxonomic distribution without (or with minimal) sequence resemblance to other organisms; and encoded by genes featuring extended intergenic regions or located in gene-sparse, repeat-rich chromosomes \parencite{Dalio2018, Todd2022}. However, as \textcite{LoPresti2015, Sperschneider2015} stress, not all secreted proteins with small size and high cysteine content necessarily serve as effectors. Conversely, fungal effectors are not universally small and cysteine-rich. Larger proteins can also function as effectors, which leads \textcite{LoPresti2015} to describe the 300 amino acid cutoff as arbitrary. Further, effector classification often relies on the absence of detectable orthologous proteins beyond the genus, but some effectors may show conservation or possess conserved functional domains \parencite{Jonge2010, Djamei2011, Mentlak2012}. Given these uncertainties in effector definition, \textcite{LoPresti2015} adopt a broad perspective, considering any secreted fungal protein as a potential effector. 

\textcite{Sperschneider2016} developed a \ac{ml} tool, EffectorP v1.0, for the identification of effectors in fungi in an attempt to combat the challenges of effector discovery explained by \textcite{Sperschneider2015, LoPresti2015}. 
EffectorP v1.0 is a Naïve Bayes \ac{ml} predictor, which was initially trained with 58 true fungal effectors from 16 species and achieved sensitivity and specificity of over 80\% \parencite{Sperschneider2016}. The negative dataset contained 14,143 proteins based on the total set of predicted secreted proteins from their 16 species (filtering the known effectors and homologs), and likely encompassed undiscovered effectors as well as non-effectors. Subsequently, EffectorP v2.0 was trained with 94 secreted true effectors from 23 species, utilising a refined negative dataset. This updated version successfully reduced effector candidates from fungal plant symbionts and saprophytes by 40\%, surpassing EffectorP v1.0 \parencite{Sperschneider2018}. The latest iteration, EffectorP (3.0), introduces a classification based on apoplastic and cytoplasmic localisation \parencite{Sperschneider2022}. EffectorP v2.0 has been employed by \textcite{Simbaqueba2020} in their analysis of \ac{Fo} f. sp. \textit{physalis}, identifying novel candidate effectors.

\Acp{mimp} provide an advantage when attempting to identify candidate effectors in \ac{Fo} genomes, as they are found upstream of all known effectors in \ac{Fol} \parencite{Schmidt2013} (see: section \ref{Chap1:fusariumEffectorome}). Indeed, \textcite{Dam2016} developed a computational pipeline (hereafter: FoEC) which identifies effectors using \acp{mimp}, to reveal effector profiles of cucurbit-infecting \ac{Fo} strains. The FoEC pipeline first identifies \acp{mimp} in a \ac{Fo} genome by searching for the \ac{mimp} \acp{tir} through regular expression. Once \acp{mimp} have been identified, FoEC expands 2500bp downstream of the \ac{mimp}, finding \acp{orf} using a custom Python script or using AUGUSTUS \parencite{Stanke2006} for gene prediction. Sequences are then filtered based on the presence or absence of a signal peptide, filtered for redundancy, and then searched against the input genome database using BLAST to create a presence/absence matrix. However, our analysis revealed that FoEC (v1) pipeline is unable to identify \acp{mimp} which have been soft-masked, a probable eventuality given that \acp{mimp} are repetitive elements. Removal of soft-masking to identify \acp{mimp} using FoEC (v1) will likely result in inaccurate gene predictions. Furthermore, FoEC (v1) only searches for effectors found downstream of a \ac{mimp}, but the study by \textcite{Schmidt2013} demonstrated that effectors may also be found upstream of a \ac{mimp}. 

FoEC has since been updated \parencite{FoEC2} (FoEC2)\footnote{Though it is not clear that the \ac{mimp} identification script has been adjusted to allow for soft-masking, it is of note that FoEC2 was developed and published following a discussion I had with the original FoEC authors.}, and uses a very similar pipeline structure. FoEC2 now also filters \acp{orf} based on cystine content and size (20aa $\leq$ size $\leq$ 600aa), clusters candidates using Diamond (v2.0.13) with Diamomnd BLASTP, MAFFT (v7.490) \parencite{Katoh2019} and uses HMMER \parencite{Eddy2011} to identify candidate effectors in the original genome database. FoEC2 now contains some of the filtering criteria which, as explained by \textcite{Sperschneider2015, LoPresti2015, Todd2022}, can miss candidates. 

Though we recognise that FoEC2 can be used as a tool for the identification of candidate effectors in \ac{Fo}, we developed a \ac{maei} pipeline\footnote{Development of the \ac{maei} pipeline was started before FoEC2 was published.} to find candidate effectors in \ac{Fo} genomes and contribute to our understanding of virulence within the \ac{FOSC}. Our pipeline considers the concerns of \textcite{LoPresti2015, Sperschneider2015, Todd2022}, instead identifying small ($\leq$450aa), secreted proteins associated with \acp{mimp} (upstream and downstream, 2500bp) in the \ac{Fo} genomes, submitting them to EffectorP v2.0 for candidate effector prediction. We applied the \ac{maei} pipeline to investigate candidate effector profiles in \ac{r1} and \ac{tr4} \ac{Focub} genomes, as well as \ac{r1} and \ac{r4} genome assemblies of \ac{Fola}. Candidate effectors identified using the \ac{maei} tool can be used as a target for molecular diagnostics, something which is currently being explored with \ac{r2}, \ac{r3}, and \ac{r4} isolates of \ac{Foa}. 

\newpage
\section{Materials and Methods}

\subsection{\textit{Fusarium} genome assembly and database generation}\label{chap3:fusariumdb}

A database of \textit{Fusarium} assemblies was produced for candidate effector identification and analysis (Table \ref{tab:GenomeDB}). Genome assemblies were generated for \ac{Fo} \ac{fsp} \textit{lactucae}\footnote{\label{note1}Included in publication currently under review.} (\acs{Fola}), \textit{apii} (\acs{Foa}), \textit{coridandrii} (\acs{Foci}), \textit{matthiolae}\footref{note1} (\acs{Foma}), \textit{narcissi}\footref{note1} (\acs{Fona}), and a \ac{Fo} isolate pathogenic towards rocket (\textit{Eruca vesicaria}) by collaborators at the \ac{niab}. Isolates were cultivated on PDA plates at 25°C for 4 days, and mycelial plugs were used to inoculate potato dextrose broth (PDB) containing streptomycin. After 4 days of incubation, mycelium was harvested, lyophilised, and stored at -80°C. DNA extraction from 20mg of lyophilised mycelium was performed using a Macherey-Nagel NucleoSpin Plant II kit (Fisher Scientific) with additional chloroform extraction after cell lysis. DNA samples were eluted in 30µl 10mM Tris HCl pH8 and analysed for quantity and purity using Nanodrop and Qubit (ratios 260/280 between 1.8-2.0, 260/230 between 2.0-2.2)  and for integrity using the Agilent TapeStation (TS 4150, Agilent Technologies). DNA samples (molecular weight >50kb) underwent Illumina PCR-free and Oxford Nanopore Technologies (ONT) long-read sequencing. Illumina sequencing was conducted as described previously \parencite{Armitage2018}. ONT sequencing utilised the ligation sequencing kit (LSK108 for \ac{Foma} AJ260, \ac{Fona} FON63; or LSK110 all \ac{Fola}, \ac{Foa}, \ac{Foci} isolates and \ac{Fo} rocket) with flow cells FLO-MIN106 R9.4 (\ac{Foma} AJ260, \ac{Fona} FON63) or FLO-MIN106 R9.4.1 (\ac{Fola}, \ac{Foa}, \ac{Foci} and \ac{Fo} rocket isolates). 

\textit{De novo} genome assemblies for \ac{Foma}, \ac{Fona}, \ac{Fola}, \ac{Foa}, \ac{Foci} and \ac{Fo} rocket isolates were constructed using ONT long-read sequence data. Quality control on ONT data employed NanoPlot v1.30.1 \parencite{DeCoster2018}. Porechop (v0.2.4) (\href{https://github.com/rrwick/Porechop}{https://github.com/rrwic\-k/Porechop}) with default settings was applied for adapter trimming, and Filtlong (v0.2.1 )(\href{https://github.com/rrwick/Filtlong}{https://github.com/rrwick/Filtlong}) removed reads shorter than 1 Kb or with a quality score less than Q9. Long-read data were assembled using NECAT (v0.0.1\_update20200803) \parencite{Chen2021} using a genome size of 50 Mb, with other parameters left as default. For long-read error correction, reads were aligned to the assemblies with Minimap2 (v2.17-r941) \parencite{Li2018} to inform one iteration of Racon v1.4.20 \parencite{Vaser2017}. This was followed by one iteration of Medaka (v1.5.0) (\href{https://github.com/nanoporetech/medaka}{https://github.com/nanoporetech/medaka}) using the r94\_min\_high\_g360 model. Illumina paired-end reads underwent quality control with FastQC (v0.11.9) (\href{https://www.bioinformatics.babraham.ac.uk/projects/fastqc/}{https://www.bioinformatics.babraham.ac.uk/projects/fastqc/}), followed by adapter and low-quality region trimming using Fastq-Mcf (v1.04) \parencite{Aronesty2013}. Alignment to long-read assemblies was conducted using Bowtie2 (v2.2.5) \parencite{Langmead2012} and SAMtools (v1.13) \Parencite{Danecek2021}. Three rounds of polishing with Pilon (v1.24) \parencite{Walker2014} corrected single base call errors and small insertions or deletions. Assembly statistics were computed using a custom Python script, and single copy ortholog analysis utilised \ac{busco} (v5.2.2) \parencite{Simao2015} with the hypocreales\_odb10 database.

Alongside the genome assemblies generated as part of this study, publicly available \textit{Fusarium} assemblies were included. Two \ac{Fs} and 12 \ac{Focub} assemblies identified following GenBank genome search (\href{https://www.ncbi.nlm.nih.gov/data-hub/genome}{https://www.ncbi.nlm.nih.gov/data-hub/genome/}) were downloaded, as well as two \ac{Focub} genome assemblies (isolate ID: 58 and 60) from the National Genomics Data Centre (NGDC), China (\href{https://ngdc.cncb.ac.cn}{https://ngdc.cncb.ac.cn/}). Further, genome assemblies for the \ac{Foa} isolates AJ720 and AJ498 have already been published by \textcite{Henry2020} so were used in this analysis for comparison. \Ac{Foa} \ac{r3} and \ac{Foci} genome assemblies generated by \textcite{Henry2020} were also included. A representative assembly (\( \leq \) 50 contigs and \(\geq \) 90\% complete BUSCOs reported) was included for other publicly available \textit{Fusarium} species and \ac{fsp}. As \textcite{Schmidt2013} reported no \acp{mimp} in the \ac{Fg} PH-1 genome, a reference assembly for \ac{Fg} PH-1 (GCA\_000240135.3) was included as an outgroup for phylogenies and negative control for \ac{mimp}-associated effector analysis. The S6, S16, and S32 genome assemblies were not included in this analysis as the raw read data were not available at the time of this analysis.

\input{Tables/GenomeDatabase}

\subsection{Phylogenetic analysis of \textit{Fusairum} isolates}


The common Fusarium genetic barcodes Tef-1\(\alpha\) and RPBII  were used to generate phylogenies of the \textit{Fusarium} assemblies included in the candidate effector analysis (See section:~\ref{chap2:phylogeny}). Briefly, homologs of each barcode were identified in each assembly in our database) using BLASTN (1e-\textsuperscript{6} cut-off). The locations of hits with greater than 70\% identity and 90\% coverage were extracted using Samtools (Version 1.15.1). Barcodes from each genome were concatenated into a single FASTA file and aligned using MAFFT (Katoh \et 2019). IQ-TREE (v2.2.0.3) (Nguyen \et 2015) was used to infer a maximum-likelihood phylogeny using the ultrafast bootstrap setting for 1000 bootstrap replicates and was visualised using iTOL (Letunic and Bork, 2021). 

\subsection{\textit{mimp}-associated effector identification (Maei) pipeline}

\subsubsection{\Ac{mimp} Identification}

Two methods of \textit{mimp} searching were developed. The first uses searching by regular expression, whereby the \textit{mimp} TIR sequences, "CAGTGGG..GCAA[TA]AA" and "TT[TA]TTGC..CCCACTG", are used as a search pattern. Where sequences matching this pattern occur within 400 nucleotides of each other a \textit{mimp} is recorded (Appendix ~\ref{apx:mimpfinditer}). 

The second method employs a Hidden Markov Model (HMM) which was developed using the HMM tool HMMER (3.3.1) (Eddy, N.D.).Publicly available \textit{mimp} sequences (Appendix \ref{apx}) and \textit{mimp} sequences identified using the regular expression method were used to build a \textit{mimp} profile-HMM. This profile-HMM was used as the input for an NHMMER search of each genome.

\subsubsection{Sequence Extraction}

Using mimps identified by both mimp searching methods, sequences 2.5kb upstream and downstream of each mimp are extracted, mereged and stored in GFF3 format. This region is also used to generate an Augustus region GFF (\ac{mimp} region + 20kb either side, to prevent truncation of gene models). The GFFs are used to generate a \ac{mimp} region only FASTA and an Augustus region only FASTA file, whereby all nucleotides outside of the \ac{mimp} or Augustus regions are hard masked.

The Augustus region fasta is then subjected to gene prediction using AUGUSTUS (3.3.3) (Stanke, et al., 2006) with the “Fusarium” option enabled, and open reading frames (ORFs) withing the mimp regions are identified using the EMBOSS (6.6.0.0) tool, getorf (https://www.bioinformatics.nl/cgi-bin/emboss/getorf) and the mimp region fasta.

The Augustus gene models and \ac{mimp} region GFF3 files were intersected and merged using BedTools \textcolor{red}{REFEREENCE and version}, to ensure that all gene models are associated with a \ac{mimp} region, but that no gene models are truncated. The intersected Augustus and \ac{mimp}-region GFF file was used as input for AGAT \textcolor{red}{REFEREENCE and version}, which extracts Augustus \ac{mimp}-associated gene models in to a FASTA file. The \ac{mimp}-associated gene model FASTA and getORF FASTA are merged to generate a \ac{mimp}-associated gene model and ORF FASTA. The output from getORF is also converted to GFF format using a custom Python script (Appendix \ref{getORF2gff.py}). These gffs were then mergered to generate a \ac{mimp}-associated gene model and ORF GFF.


\subsubsection{Candidate Effector Filtering and Prediction}

Sequences from the \ac{mimp}-associated gene model and ORF fasta were then filtered based on size, with sequences >30aa and <450aa submitted to SignalP (4.1) (Petersen, et al., 2008). Sequences which were  predicted to contain a signal peptide were parsed to EffectorP (2.0.1) (Sperschneider, et al., 2018) for effector prediction. This output is was to generate a genome-specific candidate effector sets.

\subsubsection{Candidate Effector Clustering and Distribution}

The candidate effector sets from each genome were then combined and clustered using CD-HIT (4.8.1) (Fu, et al., 2012) \textcolor{red}{percentage identity determined using command line}, to generate candidate effector clusters, whereby differences in effector profile can be determined, and specific effector clusters assessed. Data from CD-HIT was also parsed to a custom Python script, which generated an overview table and a distibution matrix.

To identify candidates which may be shared across isolates but not associated with a \ac{mimp} in all isolates, the combined candidate effector sets were then searched against the \textit{Fusarium} genomes using TBLASTN, with a cut-off 1e-6 and a percentage identity and coverage threshold of 60\%. A data matrix was generated using the TBLASTN hit data and a heatmap was generated using the R package Pheatmap. Sequences within the threshold were extracted \textcolor{red}{samtools command} and subjected to filtering using SignalP (4.1), default settings (Petersen, et al., 2008) and EffectorP (2.0.1) (Sperschneider, et al., 2018) to remove unlikely candidates. 

\textcolor{red}{Statistical significance was analysed using a Mann-Whitney U test and correlation was assessed using Spearman’s correlation in R (R version 3.6.3)}.

\subsubsection{Identification of Diagnostic Candidates}

\subsubsection{Functional Predictions and RNAseq}

\subsection{Lactcuae SIX gene stuff???}

\newpage
\section{Results}


% Plan for Figures:

% - Table of all isolates included

% - TEF phylogeny of isolates included in the assessments

% - Something on mimp ID? 

% - Ability of versions to predict effectors Table - Include FoEC.py first results?

% - Stats bar charts of assembly size and effector pred. 

% - Major heatmap of effectors 

% - Phylogeny and heatmap ggtreeExtra? 

% - Minor heatmap of race specific effectors (lactucae and banana)

% - Pred functions of race specific effectors?

% - RNA seq of lactucae 

% - Apii diagnostics 

% - Lactucae SIX gene trees??? 

\newpage
\section{Discussion}

\ac{Segmental Duplication and the Evolution of Effectors}

van Westerhoven \et (2023) investigated the effects of segmental duplications on effector repertoires.  Though previously reported by van Dam \et (2016), the authors provide a more comprehensive effector gene repertoire for \ac{Focub} and demonstrate that effector repertoires were variable among the strains. The study found that 336 out of 669 predicted effector genes in \ac{Focub4} strain II5 evolved via segmental duplications. Notably, the effector SIX1, which is essential for full virulence to banana, was found to be part of a segmental duplication. This supports van Westerhoven \et (2023) argument and suggests that segmental duplications are involved in the evolution and diversification of effector genes in \ac{Focub}, which in turn influence the pathogenicity and virulence of \textit{Fusarium} strains.

Though we know that cytoplasmic fungal effectors are secreted into plant host cells, the mechanism by which fungal pathogens deliver effector proteins into plant cells remains poorly understood. Recently, \textcite{Oliveira-Garcia2023, Wang2023} have provided evidence for the uptake of cytoplasmic effectors from \textit{Magnaporthe oryzae} and \textit{Phytophthora infestans} into host \textit{Oryza sativa} or \textit{Nicotiana benthamiana} cells, respectively, through clathrin-mediated endocytosis (CME).  

\textcolor{red}{[ALSO, IF YOU USE EFFECTORS AS A TARGET, THEY ARE PRONE TO HIGH AMOUNTS OF EVOLUTIONARY PRESSURE, AND MAY MUTATE RENDERDING DIAGNOSTICS USELESS - YOU THEREFORE NEED A MEANS OF CONSISTENTLY INDENTIFYING NEW TARGETS...]}