\section{Abstract}

\newpage
\section{Introduction}

Computational analysis has become an integral part of modern-day plant pathology. High-quality genome sequences are being generated for multiple plant and pathogen species including banana and \ac{Fo}. The \ac{ncbi} genome database currently holds 12 genome entries for the Musaceae family and 706 genome entries for \ac{Fo}; 15 of which are from \ac{Focub}, including a high-quality draft genome for \ac{Focub4} Warmington \et (2019) and \ac{Focub1} Asai \et (2019) (https://www.ncbi.nlm.nih.gov/data-hub/genome/ Accessed: 6/10/2022).  Further, novel technologies are being developed to aid in the analysis of \ac{Fo} genomes. For instance, Park \et (2010) highlight integrated platforms supporting strain identification, phylogenetics, comparative genomics and knowledge sharing for Fusarium. Sperschneider \et (2018) developed a \ac{ml} tool for the identification of effectors in fungi, which Simbaqueba \et (2020) employed in their analysis of \Fo f. sp. \textit{physalis}. The use of genomics to identify candidate effectors in \ac{Fo} is becoming commonplace and is contributing to our understanding of \ac{Fo} classification, evolution, and cryptic host specificity.

van Dam \et  (2016) developed a computational pipeline (hereafter: FoEC) which identifies effectors using \ac{mimp}s, with a view to reveal effector profiles of cucurbit-infecting \ac{Fo} strains. However, our analysis revealed that FoEC pipeline is unable to identify \ac{mimp}s which have been soft-masked, a probable eventuality given that \ac{mimp}s are repetitive elements. Removal of soft-masking to identify \ac{mimp}s using FoEC will likely result in inaccurate gene predictions downstream. Furthermore, FoEC only searches for effectors found downstream of a \ac{mimp}, but the study by Schmidt \et (2013) demonstrated that effectors may also be found upstream of a \textit{mimp}. 

We developed a \ac{mimp}-associated effector identification pipeline \ac{maei} to find effectors in \ac{Fo} and applied this pipeline to investigate effector profiles in \ac{r1} and \ac{tr4} of \ac{Focub}. Effectors identified using the \ac{maei} tool can be used as a target for molecular diagnostics and contribute to our understanding of virulence with the \ac{FOSC}. 


\textcolor{red}{[ALSO, IF YOU USE EFFECTORS AS A TARGET, THEY ARE PRONE TO HIGH AMOUNTS OF EVOLUTIONARY PRESSURE, AND MAY MUTATE RENDERDING DIAGNOSTICS USELESS - YOU THEREFORE NEED A MEANS OF CONSISTENTLY INDENTIFYING NEW TARGETS...]}

\newpage
\section{Materials and Methods}

\subsection{\textit{Fusarium} genome database and sequencing}\label{chap3:fusariumdb}
A database of publicly available Fusarium assemblies was generated for genomic analysis. Fusarium assemblies available from GenBank Genome search (\texttt{https://ww}\linebreak{w.ncbi.nlm.nih.gov/data-hub/genome/}) were downloaded, alongside two \ac{Focus} assemblies from the National Genomics Data Centra (NGDC), China (\texttt{https://ngdc.cn}\linebreak{cb.ac.cn/}). All Foc and \textit{F. sacchari }assemblies were downloaded, a representative assembly (\( \leq \) 50 contigs and a reported BUSCO of \(<98\% \)) was included for other \textit{Fusarium} species and f. spp. The \textit{Fusarium graminearum} assembly (GCA\_000240135.3) was included as an outgroup for phylogenies and negative control for effector analysis.

\subsection{Phylogenetic analysis of \textit{Fusairum} isolates}
The common Fusarium genetic barcodes Tef-1\(\alpha\) and RPBII  were used to generate phylogenies (Edel-Hermann and Lecomte, 2019) (See section:~\ref{chap2:phylogeny})Briefly, homologs of each barcode were identified in each assembly in our database) using BLASTN (1e-\textsuperscript{6} cut-off). The locations of hits with greater than 70\% identity and 90\% coverage were recorded, and extracted using Samtools (Version 1.15.1). Barcodes from each genome were concatenated into a single FASTA file and aligned using MAFFT (Katoh \et 2019). IQ-TREE (Version 2.2.0.3) (Nguyen \et 2015) was used to infer a maximum-likelihood phylogeny using the ultrafast bootstrap setting for 1000 bootstrap replicates and was visualised using iTOL (Letunic and Bork, 2021). 

\subsection{\textit{mimp}-associated effector identification (Maei) pipeline}

\subsubsection{\Ac{mimp} Identification}

Two methods of \textit{mimp} searching were developed. The first uses searching by regular expression, whereby the \textit{mimp} TIR sequences, "CAGTGGG..GCAA[TA]AA" and "TT[TA]TTGC..CCCACTG", are used as a search pattern. Where sequences matching this pattern occur within 400 nucleotides of each other a \textit{mimp} is recorded (Appendix ~\ref{apx:mimpfinditer}). 

The second method, employs a Hidden Markov Model (HMM) which was developed using the HMM tool HMMER (3.3.1) (Eddy, N.D.).Publicly available \textit{mimp} sequences (Appendix \ref{apx}) and \textit{mimp} sequences identified using the regular expression method were used to build a \textit{mimp} profile-HMM. This profile-HMM was used as the input for an NHMMER search of each genome.

\subsubsection{Sequence Extraction}

Using mimps identified by both mimp searching methods, sequences 2.5kb upstream and downstream of each mimp are extracted, mereged and stored in GFF3 format. This region is also used to generate an Augustus region GFF (\ac{mimp} region + 20kb either side, to prevent truncation of gene models). The GFFs are used to generate a \ac{mimp} region only FASTA and an Augustus region only FASTA file, whereby all nucleotides outside of the \ac{mimp} or Augustus regions are hard masked.

The Augustus region fasta is then subjected to gene prediction using AUGUSTUS (3.3.3) (Stanke, et al., 2006) with the “Fusarium” option enabled, and open reading frames (ORFs) withing the mimp regions are identified using the EMBOSS (6.6.0.0) tool, getorf (https://www.bioinformatics.nl/cgi-bin/emboss/getorf) and the mimp region fasta.

The Augustus gene models and \ac{mimp} region GFF3 files were intersected and merged using BedTools \textcolor{red}{REFEREENCE and version}, to ensure that all gene models are associated with a \ac{mimp} region, but that no gene models are truncated. The intersected Augustus and \ac{mimp}-region GFF file was used as input for AGAT \textcolor{red}{REFEREENCE and version}, which extracts Augustus \ac{mimp}-associated gene models in to a FASTA file. The \ac{mimp}-associated gene model FASTA and getORF FASTA are merged to generate a \ac{mimp}-associated gene model and ORF FASTA. The output from getORF is also converted to GFF format using a custom python script (Appendix \ref{getORF2gff.py}). These gffs were then mergered to generate a \ac{mimp}-associated gene model and ORF GFF.


\subsubsection{Candidate Effector Filtering and Prediction}

Sequences from the \ac{mimp}-associated gene model and ORF fasta were then filtered based on size, with sequences >30aa and <450aa submitted to SignalP (4.1) (Petersen, et al., 2008). Sequences which were  predicted to contain a signal peptide were parsed to EffectorP (2.0.1) (Sperschneider, et al., 2018) for effector prediction. This output is was to generate a genome-specific candidate effector sets.

\subsubsection{Candidate Effector Clustering and Distribution}

The candidate effector sets from each genome were then combined and clustered using CD-HIT (4.8.1) (Fu, et al., 2012) \textcolor{red}{percentage identity determined using command line}, to generate candidate effector clusters, whereby differences in effector profile can be determined, and specific effector clusters assessed. Data from CD-HIT was also parsed to a custom python script, which generated an overview table and a distibution matrix.

To identify candidates which may be shared across isolates but not associated with a \ac{mimp} in all isolates, the combined candidate effector sets were then searched against the \textit{Fusarium} genomes using TBLASTN, with a cut-off 1e-6 and a percentage identity and coverage threshold of 60\%. A data matrix was generated using the TBLASTN hit data and a heatmap was generated using the R package Pheatmap. Sequences within the threshold were extracted \textcolor{red}{samtools command} and subjected to filtering using SignalP (4.1), default settings (Petersen, et al., 2008) and EffectorP (2.0.1) (Sperschneider, et al., 2018) to remove unlikely candidates. 

\textcolor{red}{Statistical significance was analysed using a Mann-Whitney U test and correlation was assessed using Spearman’s correlation in R (R version 3.6.3)}.

\subsubsection{Identification of Diagnostic Candidates}

\subsubsection{Functional Predictions and RNAseq}

\subsection{Lactcuae SIX gene stuff???}

\newpage
\section{Results}

\newpage
\section{Discussion}

\ac{Segmental Duplication and the Evolution of Effectors}

van Westerhoven \et (2023) investigated the effects of segmental duplications on effector repertoires.  Though previously reported by van Dam \et (2016), the authors provide a more comprehensive effector gene repertoire for \ac{Focub} and demonstrate that effector repertoires were variable among the strains. The study found that 336 out of 669 predicted effector genes in \ac{Focub4} strain II5 evolved via segmental duplications. Notably, the effector SIX1, which is essential for full virulence to banana, was found to be part of a segmental duplication. This supports van Westerhoven \et (2023) argument and suggests that segmental duplications are involved in the evolution and diversification of effector genes in \ac{Focub}, which in turn influence the pathogenicity and virulence of \textit{Fusarium} strains.