\section{Introduction}

Computational analysis has become an integral part of modern-day plant pathology. High-quality genome sequences are being generated for multiple plant and pathogen species including banana and \Fo. The NCBI genome database currently holds 12 genome entries for the Musaceae family and 706 genome entries for \Fo; 15 of which are from Foc, including a high-quality draft genome for Foc TR4 Warmington et al., (2019) and Foc R1 Asai et al., (2019) (\texttt{https://www.ncb}\linebreak{i.nlm.nih.gov/data-hub/genome/} Accessed: 6/10/2022).  Further, novel technologies are being developed to aid in analysis \Fo genomes. For instance, Park et al., (2010) highlight integrated platforms supporting strain identification, phylogenetics, comparative genomics and knowledge sharing for Fusarium. Sperschneider et al., (2018) developed a machine learning (ML) tool for the identification of effectors in fungi, which Simbaqueba et al., (2020) employed in their analysis of \Fo f. sp. physalis. The use of genomics to identify candidate effectors in \Fo is becoming common place and is contributing to our understanding of \Fo classification, evolution, and cryptic host specificity. 
van Dam et al., (2016) developed a computational pipeline (hereafter: FoEC) which identifies effectors using \textit{\textit{mimp}s}, with a view to identify effector profiles of  cucurbit-infecting \Fo strains. However, our analysis revealed that FoEC pipeline is unable to identify \textit{\textit{mimp}} which have been soft-masked, a probable eventuality given that \textit{\textit{mimp}} are repetitive elements. Removal of soft-masking to identify \textit{\textit{mimp}} using FoEC will likely result in in-accurate gene predictions downstream. Furthermore, FoEC only searches for effectors found downstream of a \textit{mimp}, but the study by Schmidt et al., (2013) demonstrated that effectors may also be found upstream of a \textit{mimp}. 

We developed a \textit{mimp}-associated effector identification pipeline (Maei) to find effectors in \Fo and applied this pipeline to investigate effector profiles in R1 and TR4 of \Foc. Effectors identified using the Maei tool can be used as a target for molecular diagnostics and contribute to our understanding of virulence with the FOSC.  Alongside effector identification, we employed other comparative genomic techniques such as whole genome alignment and phylogenetic analysis to explore the \textit{Fusarium} genome and identify pathogen-specific regions. 

\section{Materials and Methods}

\subsection{\textit{Fusarium} genome database}\label{chap3:fusariumdb}
A database of publicly available Fusarium assemblies was generated for genomic analysis. Fusarium assemblies available from GenBank Genome search (\texttt{https://ww}\linebreak{w.ncbi.nlm.nih.gov/data-hub/genome/}) were downloaded, alongside two \Foc assemblies from the National Genomics Data Centra (NGDC), China (\texttt{https://ngdc.cn}\linebreak{cb.ac.cn/}). All Foc and \textit{F. sacchari }assemblies were downloaded, a representative assembly (\( \leq \) 50 contigs and a reported BUSCO of \(<98\% \)) was included for other \textit{Fusarium} species and f. spp. The \textit{Fusarium graminearum} assembly (GCA\_000240135.3) was included as an outgroup for phylogenies and negative control for effector analysis.

\subsection{Phylogenetic analysis of \textit{Fusairum} isolates}
The common Fusarium genetic barcodes Tef-1\(\alpha\) and RPBII  were used to generate phylogenies (Edel-Hermann and Lecomte, 2019) (See section:~\ref{chap2:phylogeny})Briefly, homologs of each barcode were identified in each assembly in our database) using BLASTN (1e-\textsuperscript{6} cut-off). The locations of hits with greater than 70\% identity and 90\% coverage were recorded, and extracted using Samtools (Version 1.15.1). Barcodes from each genome were concatenated into a single FASTA file and aligned using MAFFT (Katoh \et 2019). IQ-TREE (Version 2.2.0.3) (Nguyen \et 2015) was used to infer a maximum-likelihood phylogeny using the ultrafast bootstrap setting for 1000 bootstrap replicates and was visualised using iTOL (Letunic and Bork, 2021). 

\subsection{Development of the \textit{mimp}-associated effector identification (Maei) pipeline}
Two methods of \textit{mimp} searching were developed. The first uses searching by regular expression, whereby the \textit{mimp} TIR sequences, "CAGTGGG..GCAA[TA]AA" and "TT[TA]TTGC..CCCACTG", are used as a search pattern. Where sequences matching this pattern occur within 400 nucleotides of each other a \textit{mimp} is recorded (Appendix ~\ref{apx:mimpfinditer}). The second method, employs a Hidden Markov Model (HMM) which was developed using the HMM tool HMMER (3.3.1) (Eddy, N.D.). Briefly, publicly available \textit{mimp} sequences (Appendix X) and \textit{mimp} sequences identified using the regular expression method were used to build a \textit{mimp} profile-HMM. This profile-HMM was used as the input for an NHMMER search of each genome.
Using \textit{\textit{mimp}} identified by both \textit{mimp} finding methods, sequences 2.5kb upstream and downstream of each \textit{mimp} are extracted and subjected to prediction using AUGUSTUS (3.3.3) (Stanke, et al., 2006) with the “Fusarium” option enabled, and open reading frames (ORFs) identified using the EMBOSS (6.6.0.0) tool, getorf (https://www.bioinformatics.nl/cgi-bin/emboss/getorf). As getorf identifies the longest ORF, a custom python script was developed to extract smaller ORFs from within the ORFs found by getorf.  ORFs and predicted genes from each genome with a signal peptide (predicted using SignalP (4.1), default settings (Petersen, et al., 2008)) are then clustered using CD-HIT (4.8.1) (Fu, et al., 2012) to create a non-redundant candidate effector set for each genome. Sequences of between 30aa and 300aa are extracted from the individual, non-redundant candidate effector sets and are then clustered again using CD-HIT (80\% identity), to generate a candidate pan-effectorome. The candidate pan-effectorome is then subjected to effector prediction from EffectorP (2.0.1) (Sperschneider, et al., 2018). 
Effectors identified using the Maei pipeline were then searched back against the Fusarium genomes using TBLASTN, with a cut-off 1e-6 and a percentage identity and coverage threshold of 70\% and 90\%, respectively. A data matrix was generated using the TBLASTN hit data and a heatmap was generated using the R package Pheatmap (Figure 5). Statistical significance was analysed using a Mann-Whitney U test and correlation was assessed using Spearman’s correlation in R (R version 3.6.3). 

\subsection{Identification of pathogen-specific regions.}
To investigate the presence of pathogenicity chromosomes in \Foc, \textit{mimps}, \textit{SIX} genes and candidate effectors identified using the Maei pipeline were mapped onto the high-quality Foc (Asai et al., 2019, Warmington et al., 2019) and Fol (FOL ASSEMBLY REFERENCE) assemblies. Further, nucmer (-max match, deltafilter –g) from MUMmer (version 4.0.0rc1) (Marçais et al., 2018). was used to align the high-quality Foc (R1 and TR4), Fol and \textit{F. sacchari} genomes to help identify accessory and core regions, as well as syntenic blocks (Fokkens et al., 2020). Circos (version 0.69-8) (Krzywinski et al., 2009) was used to visualise virulence factor location and genome alignments. 