\section{Introduction}
\label{sec:Chap3Intro}

High-quality genome sequences are being generated for multiple plant and pathogen species including banana and \ac{Fo}. There are currently 17 genome entries for the Musaceae family and 736 genome entries for \ac{Fo}, 17 of which are from \ac{Focub}, in the \ac{ncbi} Genome Datasets database (\href{https://www.ncbi.nlm.nih.gov/data-hub/genome/}{https://www.ncbi.nlm.nih.gov/data-hub/genome/} Accessed: 17/01/2024), including a high-quality draft genome for \ac{Focub4} \parencite{Warmington2019} (\href{https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_007994515.1/}{GCA\_007994515.1}) and \ac{Focub1} \parencite{Asai2019} (\href{https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_005930515.1/}{GCA\_005930515.1}). Many bioinformatic tools have been developed to analyse the increasing number of genome sequences. For instance, \textcite{Park2010} highlight integrated platforms supporting strain identification, phylogenetics, comparative genomics, and knowledge sharing for \textit{Fusarium}. The use of genomics to identify \acf{ce} in \ac{Fo} is becoming commonplace and is contributing to our understanding of \ac{Fo} classification, evolution, and cryptic host specificity \parencite{Dam2016, Dam2017, Simbaqueba2020, FoEC2, Westerhoven2023}. 

Generally, fungal effectors do not share homology or conserved motifs, so computational prediction of fungal \acp{ce} is challenging \parencite{Sperschneider2022, Todd2022}. Broadly, fungal effectors have been predicted using a set of common characteristics: the presence of a secretion signal, sequence length $\leq$ 300 amino acids, and cysteine richness \parencite{Sperschneider2015}. Other features sometimes used to filter protein sets include the lack of a transmembrane domain; increased expression during host interaction; restricted taxonomic distribution without (or with minimal) sequence resemblance to other organisms; and encoded by genes featuring extended intergenic regions or located in gene-sparse, repeat-rich chromosomes \parencite{Dalio2018, Todd2022}. However, as \textcite{LoPresti2015, Sperschneider2015} stress, not all secreted proteins with small size and high cysteine content necessarily serve as effectors. Conversely, fungal effectors are not universally small and cysteine-rich. Larger proteins can also function as effectors, which leads \textcite{LoPresti2015} to describe the 300 amino acid cutoff as arbitrary. Further, effector classification often relies on the absence of detectable orthologous proteins beyond the genus, but some effectors may show conservation or possess conserved functional domains \parencite{Jonge2010, Djamei2011, Mentlak2012}. Given these uncertainties in effector definition, \textcite{LoPresti2015} adopt a broad perspective, considering any secreted fungal protein as a potential effector. 

\textcite{Sperschneider2016} developed a \ac{ml} tool, EffectorP v1.0, for the identification of effectors in fungi in an attempt to combat the challenges of effector discovery, explained by \textcite{Sperschneider2015, LoPresti2015}. EffectorP v1.0 is a Naïve Bayes \ac{ml} predictor, which was initially trained with 58 true fungal effectors from 16 species (including \ac{Fo}) and achieved sensitivity and specificity of over 80\% \parencite{Sperschneider2016}. The negative dataset contained 14,143 proteins based on the total set of predicted secreted proteins from their 16 species (filtering the known effectors and homologs), and likely encompassed undiscovered effectors as well as non-effectors. Subsequently, EffectorP v2.0 was trained with 94 secreted true effectors from 23 species, utilising a refined negative dataset. This updated version successfully reduced effector candidates from fungal plant symbionts and saprophytes by 40\%, surpassing EffectorP v1.0 \parencite{Sperschneider2018}. EffectorP v2.0 has been employed by \textcite{Simbaqueba2020} in their analysis of \ac{Fo} f. sp. \textit{physalis}, identifying novel \acp{ce}. The latest iteration, EffectorP v3.0, introduces a classification based on apoplastic and cytoplasmic localisation \parencite{Sperschneider2022}. 

\Acfp{mimp}, a non-autonomous class II \acl{te} characterised by their Terminal Invert Repeat (TIR) sequence and short length ($\sim$180–220 bp), provide an advantage when attempting to identify \acp{ce} in \ac{Fo} genomes, as they are found upstream of all known effectors in \ac{Foly} \parencite{Schmidt2013} (see: section \ref{Chap1:fusariumEffectorome}). \textcite{Dam2016} developed a computational pipeline (hereafter: FoEC) which identifies effectors using \acp{mimp} to reveal effector profiles of cucurbit-infecting \ac{Fo} strains. The FoEC pipeline first identifies \acp{mimp} in a \ac{Fo} genome by searching for the \ac{mimp} \acp{tir} through regular expression - the \ac{mimp} TIRs are used as a search term and matches in the genome assembly are recorded. Once \acp{mimp} have been identified, FoEC expands 2500bp downstream of the \ac{mimp} finding \acp{orf} using a custom Python script or using AUGUSTUS \parencite{Stanke2006} for gene prediction. Sequences are then filtered based on the presence or absence of a signal peptide, and redundancy, then searched against the input genome database using BLAST to create a presence/absence matrix. However, our analysis revealed that FoEC pipeline is unable to identify \acp{mimp} which have been soft-masked, a probable eventuality given that \acp{mimp} are repetitive elements. Removal of soft-masking to identify \acp{mimp} using FoEC will likely result in inaccurate gene predictions. Furthermore, FoEC only searches for effectors found downstream of a \ac{mimp}, but the study by \textcite{Schmidt2013} demonstrated that effectors may also be found upstream of a \ac{mimp}. 

FoEC has since been updated \parencite{FoEC2} (FoEC2)\footnote{Though it is not clear that the \ac{mimp} identification script has been adjusted to allow for soft-masking, it is of note that FoEC2 was developed and published following a discussion I had with the original FoEC authors.}, and uses a very similar pipeline structure. FoEC2 now also filters \acp{orf} based on cysteine content and size (20aa $\leq$ size $\leq$ 600aa), clusters candidates using Diamond (v2.0.13) with Diamond BLASTP, MAFFT (v7.490) \parencite{Katoh2019} and uses HMMER \parencite{Eddy2011} to identify \acp{ce} in the original genome database. FoEC2 now contains some of the filtering criteria which, as explained by \textcite{Sperschneider2015, LoPresti2015, Todd2022}, can miss candidates. 

Though we recognise that FoEC2 can be used as a tool for the identification of \acp{ce} in \ac{Fo}, we developed a \ac{maei} pipeline\footnote{Development of the \ac{maei} pipeline was started before FoEC2 was published.} to find \acp{ce} in \ac{Fo} genomes and contribute to our understanding of virulence within the \ac{FOSC}. Our pipeline considers the concerns of \textcite{Sperschneider2015, LoPresti2015, Todd2022}, identifying small ($\leq$450aa), secreted proteins associated with \acp{mimp} (upstream and downstream, 2500bp) in the \ac{Fo} genomes, submitting them to EffectorP v2.0 for \ac{ce} prediction. We applied the \ac{maei} pipeline to investigate \ac{ce} profiles in \ac{r1} and \ac{tr4} \ac{Focub} genomes, as well as \ac{r1} and \ac{r4} genome assemblies of \acf{Fola}. \acp{ce} identified using the \ac{maei} tool can be used as a target for molecular diagnostics, which is currently being explored with \ac{r2}, \ac{r3}, and \ac{r4} isolates of \acf{Foa}. 

\newpage
\section{Materials and Methods}

\subsection{\textit{Fusarium} genome database generation}\label{chap3:fusariumdb}

\subsubsection{\textit{Fusarium} genome sequencing and assembly}

A database of both publicly available and new \textit{Fusarium} genome assemblies was produced for \ac{ce} identification and analysis (Table \ref{tab:GenomeDB})\footnote{The SY-2 isolate was included, but the S6, S16, and S32 genome assemblies were not included in this database as the raw read data were not available at the time.}. Collaborators at the \acf{niab} generated genome assemblies for \ac{Fo} \acp{fsp} \textit{lactucae} (\acs{Fola}) isolates AJ520, AJ718, AJ865, AJ516, AJ592, and AJ705; \textit{apii} (\acs{Foa}) isolates AJ498 and AJ720; \textit{coridandrii} (\acs{Foci}) isolate AJ615; \textit{matthiolae} (\acs{Foma}) isolate AJ260; \textit{narcissi} (\acs{Fona}) isolate FON63; and a \ac{Fo} isolate pathogenic towards wild rocket (\textit{Diplotaxis tenuifolia}), AJ174. These isolates were cultivated on \ac{pda} plates at 25°C for 4 days, and mycelial plugs were used to inoculate \ac{pdb} containing streptomycin. After 4 days of incubation, mycelium was harvested, lyophilised, and stored at -80°C. 

DNA extraction from 20mg of lyophilised mycelium was performed using a Macher\-ey-Nagel NucleoSpin Plant II kit (Fisher Scientific) with additional chloroform extraction after cell lysis. DNA samples were eluted in 30µl 10mM Tris HCl pH8 and analysed for quantity and purity using Nanodrop and Qubit (ratios 260/280 between 1.8-2.0, 260/230 between 2.0-2.2)  and for integrity using the Agilent TapeStation (TS 4150, Agilent Technologies). DNA samples (molecular weight >50kb) underwent Illumina PCR-free and \ac{ont} long-read sequencing. Illumina sequencing was conducted as described previously \parencite{Armitage2018}. \ac{ont} sequencing utilised the ligation sequencing kit (LSK108 for \ac{Foma} AJ260, \ac{Fona} FON63; or LSK110 all \ac{Fola}, \ac{Foa}, \ac{Foci} and \ac{Fo} rocket isolates) with flow cells FLO-MIN106 R9.4 (\ac{Foma} AJ260, \ac{Fona} FON63) or FLO-MIN106 R9.4.1 (\ac{Fola}, \ac{Foa}, \ac{Foci} and \ac{Fo} rocket isolates). 

Collaborators used \Ac{ont} long-read sequence data to construct \textit{de novo} genome assemblies. NanoPlot (v1.30.1) \parencite{DeCoster2018} was used for quality control of \ac{ont} data and Porechop (v0.2.4) (\href{https://github.com/rrwick/Porechop}{https://github.com/rrwic\-k/Porechop}), with default settings, was applied for adapter trimming. Filtlong (v0.2.1)  (\href{https://github.com/rrwick/Filtlong}{https://github.com/rrwick/Filtlong}) was used to remove reads shorter than 1 Kb or with a quality score less than Q9. Long-read data were assembled using NECAT (v0.0.1\_update20200803) \parencite{Chen2021} (genome size of 50 Mb) with parameters left as default. For long-read error correction, reads were aligned to the assemblies with Minimap2 (v2.17-r941) \parencite{Li2018} and used to inform one iteration of Racon v1.4.20 \parencite{Vaser2017}. This was followed by one iteration of Medaka (v1.5.0) (\href{https://github.com/nanoporetech/medaka}{https://github.com/nanoporetech/medaka}) using the r94\_min\_high\_g360 model. The Illumina paired-end reads underwent quality control with FastQC (v0.11.9) (\href{https://www.bioinformatics.babraham.ac.uk/projects/fastqc/}{https://www.bioinformatics.\-babraham.ac.uk/projects/fastqc/}), followed by adapter and low-quality region trimming using Fastq-Mcf (v1.04) \parencite{Aronesty2013}. Alignment to long-read assemblies was conducted using Bowtie2 (v2.2.5) \parencite{Langmead2012} and SAMtools (v1.13) \Parencite{Danecek2021}. Three rounds of polishing with Pilon (v1.24) \parencite{Walker2014} corrected single base call errors and small insertions or deletions. Assembly statistics were computed using a custom Python script, and single copy ortholog analysis utilised \ac{busco} (v5.2.2) \parencite{Simao2015} with the hypocreales\_odb10 database.

\subsubsection{Publicly available \textit{Fusarium} genomes}

Two \ac{Fs} and 12 \ac{Focub} assemblies identified following GenBank Genome Database search (\href{https://www.ncbi.nlm.nih.gov/data-hub/genome}{https://ww\-w.ncbi.nlm.nih.gov/data-hub/genome/}) were downloaded, as well as two \ac{Focub} genome assemblies (isolates 58 and 60) from the \ac{ngdc}, (\href{https://ngdc.cncb.ac.cn}{https:/\-/ngdc.cncb.ac.cn/}). Further, genome assemblies for the \ac{Foa} isolates AJ720 and AJ498 have already been published by \textcite{Henry2020} (AJ720 = 207.A, GCA\_014843455.1,  AJ498 = 274.AC, GCA\_014843565.1) so were used in this analysis for comparison. \Ac{Foa} \ac{r3} (NRRL38295, GCA\_014843565.1) and a \ac{Foci} (3-2, GCA\_014843415.1) genome assemblies generated by \textcite{Henry2020} were also included. A representative assembly (\( \leq \) 50 contigs and \(\geq \) 90\% complete BUSCOs reported) was included for other publicly available \textit{Fusarium} species and \acp{fsp}. As \textcite{Schmidt2013} reported no \acp{mimp} in the \ac{Fg} PH-1 genome, a reference assembly for \ac{Fg} PH-1 (GCA\_000240135.3) was included as an outgroup for phylogenies and negative control for \ac{mimp}-associated effector analysis.

\bigskip
\input{Tables/GenomeDatabase}
\bigskip

\newpage

\subsection{Phylogenetic analysis of \textit{Fusarium} isolates}

\Ac{tef} and \ac{rbp2} phylogenies were generated as previously described (section:~\ref{chap2:phylogeny}) for the \textit{Fusarium} assemblies included in the \ac{ce} analysis. Briefly, homologues of each \ac{tef} and \ac{rbp2} were identified in each assembly in our database  (Table \ref{tab:GenomeDB}) using BLASTN (1e\textsuperscript{-6} cut-off). The locations of hits with greater than 70\% identity and 90\% coverage were extracted using SAMtools (v1.15.1). \Ac{tef} and \ac{rbp2} sequences from each genome were added to a  multiFASTA file for each barcode and aligned using MAFFT (v7.505) \parencite{Katoh2019}. IQ-TREE (v2.2.0.3) \parencite{Nguyen2015} was used to infer a maximum-likelihood phylogeny using the ultrafast bootstrap setting for 1000 bootstrap replicates and was visualised using the R \parencite{R} (v4.3.1) package ggtree (v3.10.0) \parencite{ggtree}.

\subsection{\acf{maei} pipeline}

\subsubsection{\Ac{mimp} identification}

A \acf{maei} pipeline for the identification of \acp{ce} in \ac{Fo} was developed (Figure: \ref{fig:MaeiPipeline}). The \ac{maei} pipeline first identifies \acp{mimp} in \textit{Fusarium} genomes. Two methods of \textit{mimp} searching were developed. The first identifies \acp{mimp} with a custom Python (v3.8.8) script \parencite{Python} which uses the Biopython (v1.78) package \parencite{biopython}. As performed in \textcite{Schmidt2013, Dam2016, Armitage2018} \acp{mimp} were identified in each \textit{Fusarium} genome using regular expression, whereby the \textit{mimp1-4} consensus \ac{tir} sequences \parencite{Bergemann2008, Schmidt2013}, "CAGTGGG..GCAA[TA]AA" and "TT[TA]TTGC..CCCACTG", were used as a search pattern. Where sequences matching this pattern occurred within 400 nucleotides of each other a \ac{mimp} was recorded (script available via \href{https://github.com/JamiePike/Maei/blob/main/bin/Mimp_finditer.py}{GitHub}). The second method employs a \acf{hmm}, which was developed using the \ac{hmm} tool HMMER (v3.3.1) \parencite{Eddy2011}. Publicly available \ac{mimp} sequences (Appendix B\ref{apx:RefMimps}) and \ac{mimp} sequences identified using the regular expression method were used to build a \ac{mimp} profile-\ac{hmm}. This profile-\ac{hmm} was used as the input for an NHMMER (v3.3.1) \parencite{Eddy2011} search of each genome (full command-line arguments available via GitHub: \href{https://github.com/JamiePike/Maei/blob/main/dev/notes/Mimp_Searching_Methods.md}{https://github.com/JamiePike/Maei/dev/notes/M\-imp\_Searching\_Methods.md}).

\subsubsection{Sequence extraction}

Using \acp{mimp} identified by both \ac{mimp} searching methods, sequences 2.5 kb upstream and downstream of each \ac{mimp} were extracted, merged (to create a non-redundant set), and stored in GFF3 format using the BEDtools (v2.25.0) suite \parencite{Quinlan2010} (full command-line arguments available on GitHub: \href{https://github.com/JamiePike/Maei/blob/main/bin/Maei_v5.sh}{https://github.com/JamiePike/Maei/bl\-ob/main/bin/Maei\_v5.sh}). Using the GFF3 locations, two FASTA files were generated i) "\ac{mimp}-region" FASTA where all nucleotides not within a 5-kb window (2.5 kb upstream + 2.5 kb downstream) of a \ac{mimp} were hard-masked, ii) "AUGUSTUS-region" where all nucleotides outside of a 45-kb window (22.5 kb upstream + 22.5 kb downstream) of a \ac{mimp} were hard-masked. The "AUGUSTUS-region" FASTA file was generated with a 45-kb window to prevent truncation of gene models. 

\Acp{orf} withing the "\ac{mimp}-regions" were identified using the EMBOSS (6.6.0.0) tool, getorf (https://www.bioinformatics.nl/cgi-bin/emboss/getorf). The "AUGUSTUS-region" FASTA was subjected to gene prediction using AUGUSTUS (3.3.3) \parencite{Stanke2006} with the “Fusarium” option enabled. The AUGUSTUS (v3.3.3) gene models and "\ac{mimp}-region" GFF3 files were intersected and merged using the BEDtools (v2.25.0) suite \parencite{Quinlan2010} to ensure that gene models which may be partially within the 5kb "\ac{mimp}-region" window are not truncated, but that gene models outside of the 5kb "\ac{mimp}-region" window but within the 45kb "AUGUSTUS-region" window are not included. The intersected AUGUSTUS (v3.3.3) gene models and "\ac{mimp}-region" GFF3 file was used as input for AGAT (v1.0.0) \parencite{DainatN.D.}, which was used to extract AUGUSTUS (v.3.3.3) \ac{mimp}-associated gene models in FASTA file format. The \ac{mimp}-associated gene model FASTA file and getORF FASTA were merged to generate a \ac{mimp}-associated gene model and \ac{orf} FASTA file. The output from getORF was also converted to GFF3 format using a custom Python script (available via \href{https://github.com/JamiePike/Maei/blob/main/bin/getORF2bed.py}{ GitHub https://githu\-b.com/JamiePike/Maei/blob/main/bin/getORF2bed.py}). These GFFS were then merged to generate a \ac{mimp}-associated gene models and \acp{orf} GFF3 and FASTA file for each genome using the BEDtools (v2.25.0) suite.

\subsubsection{\ac{ce} filtering and prediction}

Sequences from the \ac{mimp}-associated gene model and ORF FASTA were then filtered based on size using SAMtools (v1.6) and the BEDtools (v2.25.0) suite, with sequences $\ge30$ aa and $\le450$ aa submitted to SignalP (v5.0b) \parencite{Petersen2011}. Sequences that were predicted to contain a signal peptide were passed to EffectorP (v2.0.1) \parencite{Sperschneider2018} for \ac{ce} prediction. This output was used to generate genome-specific, \ac{mimpce} sets.

\subsubsection{\Acl{cec}ing and distribution}

The genome-specific \ac{mimpce} sets were then combined and clustered using CD-HIT (v4.8.1) \parencite{Fu2012} at 80\% identity to generate \acfp{cec}. Data from CD-HIT (v4.8.1) was also passed to a custom Python script, which was used to generate an overview table and a distribution matrix (available via GitHub: \href{https://github.com/JamiePike/Maei/blob/main/bin/Processingcdhit.py}{https://github.com/Jamie\-Pike/Maei/blob/main/bin/Processingcdhit.py}).

To identify candidates which may be shared across isolates but not associated with a \ac{mimp} in all isolates, the combined genome-specific \ac{mimpce} sets were then searched against the \textit{Fusarium} genomes using TBLASTN, with a cut-off 1e-6 and a percentage identity and coverage threshold of 65\%. Sequences within the threshold were extracted using SAMtools (v1.6) and the BEDtools (v2.25.0) suite and were subjected to filtering using SignalP (v5.0b), default settings and EffectorP (v2.0.1) to remove unlikely candidates. The remaining sequences (hereafter: \acfp{ce}) were clustered using CD-HIT (v4.8.1) \parencite{Fu2012} at 65\% identity to generate \acp{cec}. To identify differences in the distribution of \acp{cec} among assemblies, a binary presence/absence matrix was generated, where presence ("1") for a given isolate was counted if $\geq1 $ \ac{ce} clustered within a \ac{cec}.

Pearson’s correlation coefficient was calculated in R (v4.3.1) \parencite{R} to investigate the relationship between genome size, \ac{mimp}, and \ac{ce} content. The distribution of \acp{cec} across \textit{Fusarium} genome assemblies was recorded in a binary presence/absence matrix and hierarchical clustering of \ac{cec} distribution and visualisation was performed using the R (v4.3.1) \parencite{R} package, ComplexHeatmap (v2.15.4)  \parencite{ComplexHeatmap} setting the clustering distance for rows and columns to "binary". UpSet plots were generated using the R (v4.3.1) \parencite{R} package ggupset (v0.3.0) \parencite{ggupset}, and \ac{cec} distribution heatmaps with \ac{tef} phylogenies were generated for \ac{Focub}, \ac{Foci} and \ac{Foa}, as well as \ac{Fola} using the R (v4.3.1) \parencite{R} package ggtreeExtra (v1.13.0) \parencite{ggtree}. The full R (v4.3.1) script is available in Markdown format in the GitHub repository associated with the project (\href{https://github.com/JamiePike/Maei/blob/main/exp/AllFusAnalysis-Chap3/R/AnalysisCandidateEffectorSets.md}{https://github.com/JamiePike/Maei/blob/mai\-n/exp/AllFusAnalysis-Chap3/R/AnalysisCandidateEffectorSets.md}) 

\begin{figure}[htp!]
    \centering
    \includegraphics[width=14cm]{Figures/Maie_v5_Figure.pdf}
    \caption[Overview of the \acf{maei} pipeline.]{\textbf{Overview of the \acf{maei} pipeline.}}  
    \label{fig:MaeiPipeline}
\end{figure}

\subsection{Identification of reciprocal best hits \acl{tnau} \aclp{ce} against \textit{Fusarium} banana pathogens}

The MMseq2 easy-rbh (v13.45111) \parencite{Steinegger2017} was used to identify reciprocal best hits of \acp{ce} identified in the \ac{tnau} genome assemblies for isolates, S6, S16, S32, and SY-2, against \ac{ce} sets identified as part of the \ac{maei} pipeline from \textit{Fusarium} pathogenic towards banana ($\geq 20\%$ identity, $\geq 45\%$ alignment length, $\geq$ 30 bitscore). Full command line arguments can be found in the associated documents of the \ac{maei} GitHub repository (\href{https://github.com/JamiePike/Maei}{https://github.com/JamiePike/Maei}).   

\subsection{Identification and phylogenetic analysis of \aclp{sixg} in \acl{Fola}}

The set of homologues for \acp{sixg} was determined in all genomes of \ac{Fola} and other \textit{F. oxysporum} assemblies, produced in collaboration with \ac{niab} and or publicly available, for the \textit{F. oxysporum} endophyte Fo47, and for \ac{Fo} \acp{fsp} \textit{cepae, conglutinans, coriandrii, cubense, lini, lycopersici, matthiolae, niveum, rapae}, and \textit{vasinfectum} obtained from GenBank through a genome search (\href{https://www.ncbi.nlm.nih.gov/data-hub/genome/}{https://www.ncbi.nlm.nih.gov/data-hub/genome/}) (Table \ref{tab:GenomeDB}). Reference sequences for SIX1-SIX15 from \ac{Foly} (isolate 4287) were retrieved from the NCBI database (Appendix B\ref{apx:sixgenerefs}), and homologues of each \ac{sixg} were identified in each assembly using tBLASTx (1e\textsuperscript{-6} cut-off). A binary data matrix indicating presence (“+”) or absence (“-”) was generated based on the tBLASTx hit data. Phylogenies for \acp{sixg} were then constructed for the \acp{sixg} present in \ac{Fola} \ac{r1} (\textit{SIX9, SIX14}) and \ac{Fola} \ac{r4} (\textit{SIX8, SIX9, SIX14}). The positions of tBLASTx hits for \textit{SIX8}, \textit{SIX9}, and \textit{SIX14} were recorded (1e-\textsuperscript{6} cut-off), and the sequence within these regions was extracted using SAMtools (v1.15.1). Extracted regions from each genome were added to a multiFASTA file for each \ac{sixg}. Multiple sequence alignment was performed using MAFFT (v7.505) \parencite{Katoh2019}, with the “—adjustdirectionaccurately” and “–reorder” options. To ensure accurate alignment, any overhanging regions were manually inspected and trimmed. Maximum-likelihood phylogenies were inferred using IQ-TREE (v2.2.0.3) \parencite{Nguyen2015} with the ultrafast bootstrap setting for 1000 bootstrap replicates, and the resulting trees were visualised using the R (v4.3.1)  \parencite{R} package, ggtree (v3.10.0) \parencite{ggtree}.


\subsection{Expression and putative function of \aclp{ce}}

\subsubsection{\Acf{Fola} inoculation, RNA extraction and RNAseq analysis}

Collaborators at \ac{niab} inoculated lettuce (\textit{Lactuca sativa}) seedlings with \ac{Fola} \ac{r1} and \ac{r4}, as well as non-pathogenic \ac{Fo} isolates and conducted RNA extraction and analysis. Their method was adapted from \textcite{Taylor2016}. Two variants of \ac{Fola} \ac{r4} (AJ516, AJ705) and \ac{Fola} \ac{r1} AJ520 were used, along with controls: \ac{Fo} \ac{fsp} \textit{matthiolae} AJ260 (pathogenic on column stocks) and \ac{Fo} Fo47 (non-pathogenic isolate) (isolates indicated in Table \ref{tab:GenomeDB}). A non-inoculated control treatment was also established. Autoclaved ATS medium (5 mM \ch{KNO3}, 2.5 mM \ch{KPO4}, 3 mM \ch{MgSO4}, 3 mM \ch{Ca(NO3)^2}, 20 mM Fe-EDTA, 70 mM \ch{H3BO3}, 14 mM \ch{MnCl2}, 0.5 mM \ch{CuSO4}, 1 mM \ch{ZnSO4}, 0.2 mM \ch{Na2MoO4}, 10 mM NaCl, 0.01 mM \ch{CoCl2}, 0.45\% Gelrite) (Duchefa Biochemie, Haarlem, The Netherlands) in square petri dishes (12 x 12 x 1.7 cm, Greiner Bio-One, UK) was used for growth. \ac{Fola}-susceptible lettuce seeds (cv. Kordaat) were surface-sterilised (10\% bleach water (v/v) solution for 5 min and then rinsed three times with \ac{sdw}) and placed on agar plates. Stacks of plates were chilled for 4 days, followed by incubation at 15°C for 8 days and then 25°C for 6 days. Conidial suspensions of each two-week old \ac{Fo} culture (grown on \ac{pda} at 25°C) was prepared, adjusted to 1 x 10\textsuperscript{6} spores mL\textsuperscript{-1} in \ac{sdw} with the addition of 200 \(\mu\)L of Tween20 L\textsubscript{-1}, and 1.5mL pipetted directly onto lettuce roots (seven agar plates per isolate). Non-inoculated controls received \ac{sdw} + Tween. RNAseq samples were obtained from lettuce roots at 96 \ac{hpi} (four agar plates per isolate) and pooled into one sample of 12 plants per plate, corresponding to the peak expression of \textit{SIX8}, \textit{SIX9}, and \textit{SIX14} (data not shown). Lettuce roots were rinsed in \ac{sdw}, flashfrozen in liquid \ch{N2} and stored at -80°C before extraction. Disease assessment was conducted two \ac{wpi} (using three remaining plates per isolate).

In order to identify unregulated genes \textit{in planta}, \ac{Fo} isolates were cultured \textit{in vitro}. Spore suspensions (500 \(\mu\)L 1 x 10\textsuperscript{6} spores mL\textsuperscript{-1}) were pipetted onto \ac{pda} plates (prepared as above) with autoclaved cellulose discs and incubated for 96 hours at 25°C. Mycelium was harvested by scraping off the layer growing on the cellulose surface followed by flash freezing. 

Lettuce roots were ground to a fine powder using a pestle and mortar filled with liquid nitrogen and approximately 100 mg of tissue transferred to a 2 mL tube. RNA was extracted using Trizol® reagent (Thermo Fisher Scientific). Extracted RNA was precipitated using 900 \(\mu\)L of lithium chloride to 100 \(\mu\)L of RNA (250 \(\mu\)L LiCl2 + 650 \(\mu\)L DEPC treated water). DNase1 (Sigma-Aldrich) was used to remove DNA. To check degradation, RNA samples were visualised on a 2\% agarose gel (containing GelRedTM at 2 \(\mu\)L per 100 mL of gel) with Orange G (Sigma-Aldrich) loading dye. RNA samples were sent to  Novogene for polyA-enrichment, followed by Illumina PE150 sequencing at a depth of 23G raw data per sample for \textit{in planta} samples and 9G per sample for \textit{in vitro} mycelial RNA samples.

Transcripts from both \ac{Fo} isolates and lettuce RNA-Seq reads had adapters trimmed using Fastq-Mcf (v1.04) \parencite{Aronesty2013}. To distinguish \ac{Fo} reads from lettuce, the lettuce genome (\textit{Lactuca sativa}, \ac{ncbi} accession: GCF\_002870075.3) was obtained from \ac{ncbi}. Pre-processed reads were then aligned to respective \ac{Fo} assemblies using \ac{star} (v2.7.10) \parencite{Dobin2013}, incorporating the flag --outReadsUnmapped to generate a file containing non-mapping reads. The non-mapping putative \ac{Fo} RNA-Seq reads were pseudo-aligned to the respective reference genome and quantified using Kallisto (v0.48.0) \parencite{Bray2016}. Subsequently, the R (v4.1.3) \parencite{R} package, DESeq2 (v1.34.0) \parencite{Love2014} was used to conduct differential gene expression analysis. The contrast function was applied to produce a list of deferentially expressed genes. All genes up-regulated \textit{in planta} (log 2 \ac{fc} $\ge2.0$ compared to \ac{pdb} grown controls) were identified for each isolate. Up-regulated genes were then filtered based on the presence/absence of a signal peptide (gene length $\le1$kb). Genes identified as CAZYmes, secondary metabolites, transposons or those with homologues of greater than 70\% identity in the non-pathogen Fo47 were discarded. The up-regulated genes remaining were then sorted based on presence within 2.5 kb of a \ac{mimp}, level of induction based on log 2 \ac{fc}, and presence on an accessory contig. Expressed \acp{ce} were grouped by orthogroup (data not shown\footnote{Publication under review, \textcite{FolaManuscript}}) and all members of the orthogroup were cross-referenced for level of expression. 

\subsection{Reciporcal search of \ac{tnau} \aclp{ce} against selected \textit{Fusarium} wilt of banana \ac{mimp}-associated \ac{ce} sets}

Using the \acp{ce} identified in the  \ac{tnau} genome assemblies (S6 CEs = 333, S16(CEs = 289, S32 CEs = 314, and SY-2 CEs = 289) (see section \ref{tnauCEs}), we identified reciprocal best hits in the \ac{mimp}-associated \ac{ce} sets form selected \ac{Fusarium} wilt of banana genomes: \ac{Focub} UK0001, (CEs = 127), 160527 (CEs = 95), VPR144083 (CEs = 50), VPR144084 (CEs = 104), and \ac{Fs} isolate FS66 (CEs = 12).  Reciprocal best hits were determined utilising MMseqs2 easy-rbh (v13.45111) (\href{https://github.com/soedinglab/MMseqs2}{https://github.com/soedinglab/MMseqs2}),
with a minimum threshold of 20\% identity, 45\% alignment length, and a bitscore of at least 30.

\subsection{Data and software availability}

The complete computational pipelines, along with command-line arguments and bash, R, and Python scripts employed for the \ac{cec} identification and analyses, can be accessed in the Maei GitHub Repository (\href{https://github.com/JamiePike/Maei}{https://github.com/JamiePike/Maei}). Previous versions and supporting development documentation is available in Markdown format for \ac{maei} \href{https://github.com/JamiePike/Maei/tree/main/dev}{pipeline development}.

\newpage
\section{Results}

\subsection{\ac{tef} phylogeny of \textit{Fusarium} isolates included in candidate effector analysis}

Host specificity in \ac{Fo} is often attributed to the compartmentalisation of the \ac{Fo} genome, whereby chromosomes that  encode functions
necessary for primary metabolism and reproduction are designated \acfp{cc}, and chromosomes encoding virulence genes and possessing a high number of \acp{te} are designated \acfp{ac} \parencite{Ma2010, Dam2017}. A phylogenetic analysis using the common \ac{Fo} \ac{cc} barcode, \acf{tef}, was conducted to complement our effector analysis (Figure \ref{fig:TEF1aPhyloaMaei}). Two distinct clades emerged, the first containing the \textit{Fusairum} isolates, S16, S32, and SY-2 from \ac{tnau} as well as the \acl{Fs} and \acl{Fm} isolates (pink clade, Group 1, Figure \ref{fig:TEF1aPhyloaMaei}). The second clade (yellow clade, Figure \ref{fig:TEF1aPhyloaMaei}) contained the \ac{Fo} isolates and was separated into seven \ac{tef} groups.  

Isolates from the same \ac{fsp} did not consistently fall into the same \ac{tef} groups. The \ac{Focub} isolates were separated across three \ac{tef} groups, the first 
contained \ac{Focub4} isolates, and VPR144081, VPR144082, and VPR144083 (Group 2, Figure \ref{fig:TEF1aPhyloaMaei}); the second contained \ac{Focub1} isolate 160527, the \ac{tnau} isolate S6, as well as VPR144079 and VPR144084; the third \ac{Focub} \ac{tef} group contained the \ac{Focub1} isolates N2 and 60 (Group 6, Figure \ref{fig:TEF1aPhyloaMaei}). The \ac{Foa} isolates also fell into separate \ac{tef} groups. The first contained \ac{Foa} \ac{r3} (NRRL38295) and \ac{r4} isolates (AJ498 and 274AC) (Group 8, Figure \ref{fig:TEF1aPhyloaMaei}), and the second contained \ac{Foa} \ac{r2} isolates (AJ720 and 207A) (Group 5, Figure \ref{fig:TEF1aPhyloaMaei}). Like \ac{Foa} and \ac{Focub}, \ac{Foci} isolates were divided across multiple \ac{tef} groups (Group 7 and Group 8, Figure \ref{fig:TEF1aPhyloaMaei}). Interestingly, the \ac{Fola} isolates all sat within the same \ac{tef} group (Group 5, Figure \ref{fig:TEF1aPhyloaMaei}), alongside \ac{Foma} isolate AJ620 and \ac{Foa} \ac{r2} isolates (AJ720 and 207A). 

\begin{figure}[hp!]
    \centering
    \includegraphics[width=12cm]{Figures/BasicTEFPhylo.png}
    \captionsetup{width=\textwidth}
    \caption[Maximum likelihood phylogeny from alignment of \Acl{tef} sequences from \textit{Fusarium} isolates included in candidate effector analysis.]{\textbf{Maximum likelihood phylogeny from alignment of \Acf{tef} sequences from \textit{Fusarium} isolates included in candidate effector analysis.} The tree is rooted through \textit{Fusarium graminearum} PH-1 \ac{tef}, which has had the branch length adjusted by 0.03. Branch lengths show expected number of substitutions per site. White boxes indicate bootstrap values from 1000 bootstrap replicates. \acl{Fs} clade is shown in light pink, the \ac{Fo} clade is shown in yellow. IQ-TREE2 (v2.2.0.3) determined best model of substitution was TNe+G4. np indicates non-pathogen.}
    \label{fig:TEF1aPhyloaMaei}
\end{figure}

\subsection{Relationship between \acp{mimp}, \aclp{ce} and genome assembly size}


To investigate differences in effector repertoire between \textit{Fusarium}, we used \acp{mimp}, which are a short ($\sim180 - 220$ bp), transposable element commonly associated with effectors in \ac{Fo}. \Acp{mimp} (n=11,652) and \acfp{ce} (n=8,678) were identified in all 42 genome assemblies included in the \ac{maei} analysis (Table \ref{tab:GenomeDB}). The number of \acp{mimp} ranged from one in the \ac{Fg} PH-1 assembly to 675 in the assembly for \ac{Foci} AJ615, with an average of 277 identified across all genome assemblies (Table \ref{tab:CandEffNo}). \Acp{ce} showed a comparable range, with only six identified in the \ac{Fg} PH-1 assembly and 603 identified in \ac{Foci} AJ615, averaging 206 per assembly. Assemblies with more \acp{mimp} tended to have higher counts of \acp{ce}. Pearson's correlation coefficient revealed a strong positive relationship between \acp{mimp} and \acp{ce} (r(40) = 0.88, p < 0.01). 

\input{Tables/CandidateEffectorNumber}

To explore the distribution of \acp{ce} within \acp{fsp}, the \ac{maei} analysis incorporated multiple assemblies of \ac{Focub} (n=14), \ac{Fola} (n=6), \ac{Foa} (n=5), and \ac{Foci} (n=2). In \ac{Focub}, the average count of \acp{mimp} was 137, while the average count of \acp{ce} was 77. \ac{Foci} exhibited higher averages, with 577 \acp{mimp} and 459 \acp{ce}. \ac{Fola} and \ac{Foa} displayed intermediate values, with \ac{Fola} averaging 565 \acp{mimp} and 403 \acp{ce}, and \ac{Foa} 366 \acp{mimp} and 361 \acp{ce}. Notably, the larger genome assemblies tended to have higher \ac{mimp} and \ac{ce} counts. Pearson's correlation coefficient revealed that there was a significant positive correlation between genome assembly size and \ac{mimp} content (r(40) = 0.87, p < 0.01) and between genome assembly size and \ac{ce} content (r(40) = 0.91, p < 0.01) (Figure \ref{fig:MaeiStats}). 

\begin{figure}[h!]
    \centering
    \includegraphics[width=13cm]{Figures/StatsOverview.png}
    \caption[Overview of \ac{ce} and \ac{mimp} content in relation to genome assembly size.]{\textbf{Overview of \ac{ce} and \ac{mimp} content in relation to genome assembly size.} A strong positive correlation between \ac{mimp} and \ac{ce} content (r(40)=0.88, p\textless0.01) was observed. Additionally, there is a significant positive correlation between genome assembly size and \ac{ce} content (r(40)=0.91, p\textless0.01), and between genome assembly size and \ac{mimp} content (r(40)=0.87, p\textless0.01).}
    \label{fig:MaeiStats}
\end{figure}

The average number of \acp{mimp} and \acp{ce} in different races of the same \ac{fsp} was also investigated. For the \ac{Fola} genome assemblies, the average number of \acp{mimp} was comparable, standing at 546 for \ac{r1} (n=3) and 583 for \ac{r4} (n=3). However, there was a notable disparity in the average counts of \acp{ce}, with 300 identified in the \ac{Fola} \ac{r1} genome assemblies (range: 260 to 343), compared to an average of 507 in the \ac{r4} assemblies (range: 482 to 548) (Figure \ref{fig:MaeiBoxPlot}). In \ac{Focub}, the difference in the number of  \acp{mimp} and \acp{ce} between races was less pronounced, with averages of 121 and 134 \acp{mimp}, and 68 and 83 \acp{ce} identified in the \ac{r1} (n=3) and \ac{tr4} (n=6) genome assemblies, respectively.

\begin{figure}[hp!]
    \centering
    \includegraphics[width=\textwidth]{Figures/MimpsAndCandEffs_FspOfInterest.png}
    \caption[Boxplot of \ac{ce} and \ac{mimp} content difference races of \ac{Foa}, \ac{Focub}, and \ac{Fola}.]{\textbf{\Acf{mimp} and \acf{ce} content differs between races of \ac{Foa}, \ac{Focub}, and \ac{Fola}.} Black lines denote race median, whiskers indicate minimum and maximum, while boxes the first and third quartile. Only one genome assembly was included for \ac{Foa} \ac{r3}. The total number of genome assemblies per race was not equal; \ac{Foa} \ac{r2} = 2, \ac{r3} = 1, \ac{r4} = 2; \ac{Focub} \ac{r1} = 3,  \ac{tr4} = 6; \ac{Fola} \ac{r1} = 3, \ac{r4} = 3.}
    \label{fig:MaeiBoxPlot}
\end{figure}

\subsection{\textit{Fusarium} genomes from the same \textit{formae speciales} have similar \acl{cec} profiles}

We identified 325 \acfp{cec} by grouping \acp{ce} identified using the \ac{maei} pipeline with $\ge65\%$ identity. On average, \acp{cec} contained 27 \acp{ce}, though the total number of sequences in a \ac{cec} ranged from 895 (n=1, CEC\_234) to one (n=80). A binary presence/absence matrix revealed that the total number of \acp{cec} varied among isolates (Table \ref{tab:CandEffNo}). The six \acp{ce} identified in the \ac{Fg} PH-1 genome assembly were distributed across six \acp{cec}, whereas the 603 \acp{ce} identified in the \ac{Foci} AJ615 genome assembly were distributed among 106 \acp{cec} (Table \ref{tab:CandEffNo}). CEC\_80 emerged as a 'core \ac{cec}' for \textit{Fusarium}, as it was conserved across all \textit{Fusarium} genome assemblies, while eight core \acp{cec} were found among the \ac{Fo} genome assemblies. \ac{Focub} had an average of 45 \acp{cec} per genome assembly (range: 34 to 56, Table \ref{tab:CandEffNo}), which is low compared to the average 83 \acp{cec} for \ac{Fola} (range: 76 to 90), and 102 \acp{cec} for \ac{Foci}. Notably, only two \ac{Foci} genome assemblies were included (isolate AJ615, CEC=106; isolate 3-2, CEC=99, Table \ref{tab:CandEffNo}). 

Hierarchical clustering of \textit{Fusarium} genomes using the  \ac{cec} presence/absence matrix revealed that \ac{Fo} genomes tended to group within their respective \ac{fsp} (Figure \ref{fig:MaeiHeatmap}). Notable exceptions were observed for \ac{Foa} and \ac{Foci}. \ac{Foa} \ac{r2} clustered separately from \ac{Foa} \ac{r3} and \ac{r4}. Similarly, the \ac{Foci} isolates grouped separately based on \ac{cec} profile. The \ac{cec} profile of \ac{Foa} \ac{r3} and \ac{r4} genome assemblies was nearly identical, and was similar to that of \ac{Foci} 3-2 (Figure \ref{fig:MaeiHeatmap}). The \ac{Foa} \ac{r3} and \ac{r4} isolates formed a monophyletic \ac{tef} group in the \ac{tef} phylogeny (\ac{tef} Group 8, Figure \ref{fig:TEF1aPhyloaMaei}, \ref{fig:MaeiHeatmap}, and \ref{fig:Maei_celeryandcoriander}). \ac{Foa} contained a set of 54 core \acp{cec} and \ac{Foci} had 64 core \acp{cec}. A common set of 85 \acp{cec} were identified in \ac{Foa} \ac{r3} and \ac{r4}. \ac{Foci} 3-2 shared a total of 73 \acp{cec} with \ac{Foa} \ac{r3} and \ac{r4}, nine more \acp{cec} than it shared with the other \ac{Foci} isolate, AJ615 (Figure \ref{fig:UpSetCECofFoa}). 

\begin{sidewaysfigure}[hp!]
    \centering
    \includegraphics[width=\textwidth]{Figures/EffectorsHeatmap.png}
    \captionsetup{width=24cm}
    \caption[Heatmap of \acl{cec}s distribution in \textit{Fusarium} species.]{\textbf{Heatmap of \acf{cec} distribution in \textit{Fusarium} species.} \Acp{cec} were determined using CD-HIT (v4.8.1) at 80\% identity, following \ac{ce} identification using TBLASTN, (cut-off 1e-6 65\% identity and coverage threshold) and filtering using SignalP (v5.0b) and EffectorP (v2.1.0). Tef1a group denotes the \ac{tef} group assigned in Figure \ref{fig:TEF1aPhyloaMaei}. np indicates non-pathogen.}
    \label{fig:MaeiHeatmap}
\end{sidewaysfigure}

\begin{sidewaysfigure}[hp!]
    \centering
    \includegraphics[width=0.8\textwidth]{Figures/HeatmapAndPhylo_ApiiAndCoriandriiOnly.png}
    \captionsetup{width=20cm}
    \caption[CEC profiles of \ac{Fola} isolates ordered by \ac{tef} maximum likelihood phylogeny.]{\textbf{CEC profiles of \ac{Fola} isolates ordered by \ac{tef} maximum likelihood phylogeny.} Branch length shows the expected number of substitutions per site. White boxes indicate bootstrap values from 1000 bootstrap replicates. IQ-TREE2 (v2.2.0.3) best model of substitution; TNe+G4. \Acfp{fsp} and isolate codes are indicated at tree tips. \Acp{cec} were determined using CD-HIT (v4.8.1) at 80\% identity, following \ac{ce} identification using TBLASTN, (cut-off 1e-6 65\% identity and coverage threshold) and filtering using SignalP (v5.0b) and EffectorP (v2.1.0). \ac{Fo} isolate Fo47 was included for \ac{cec} profile comparison against non-pathogen. np indicates non-pathogen. \ac{Foci} GL306 was not included in the \ac{Maei} analysis.}
    \label{fig:Maei_celeryandcoriander}
\end{sidewaysfigure}


\begin{sidewaysfigure}[hp!]
    \centering
    \includegraphics[width=\textwidth]{Figures/UpSetCECofApii.png}
    \captionsetup{width = 22cm}
    \caption[Upset plot of \ac{cec} distribution between \ac{Foa} and \ac{Foci}.]{\textbf{Upset plot showing the distribution of \acp{cec} sets identified using the \ac{maei} pipeline in \ac{Foa} and \ac{Foci}.} \ac{Foa} \ac{r2} is shown in yellow, \ac{r3} is shown in light blue,  and \ac{r4} is shown in dark blue. \ac{Foci} is shown in grey. The \ac{Fo} endophyte, Fo47, is shown in brown. Each vertical line represents a set with intersections indicated by circles. The top bar plot records intersection size (number of shared CECs). The bar plot on the right-hand side counts the size of each set (total number of CECs identified in the assembly). The plot was generated using the ggupset (v0.3.0) \parencite{ggupset} package in R (v4.3.1) \parencite{R}.}
    \label{fig:UpSetCECofFoa}
\end{sidewaysfigure}


\subsection{\Acl{ce} complement of \textit{Fusarium} isolates pathogenic towards banana.} 

 Three \acp{mimp} and 12 \acp{ce} were identified in the genome assembly for the \ac{tnau} isolate, SY-2 (Table \ref{tab:CandEffNo}). The \ac{cec} profile of SY-2 was similar to that of the \ac{Fs} genome assemblies for isolates FS66 and NRRL\_66326 (Figure \ref{fig:MaeiHeatmap}). Notably, these genome assemblies (SY-2, FS66, and NRRL\_66326) formed a distinct \ac{cec} profile group separate from the \ac{Fo} species complex (FOSC) (Figure \ref{fig:MaeiHeatmap}). This pattern mirrors the \ac{tef} phylogeny, where SY-2 and \ac{Fs} sit in a distinct clade from the genomes within the \ac{FOSC} (Group 1, Figure \ref{fig:TEF1aPhylo}). Interestingly, the number of \acp{cec} in \ac{Fs} (FS66 = 12 and NRRL\_66326 = 15) and SY-2 was consistently less than the number of \acp{cec} identified in the non-pathogenic \ac{Fo} genome assemblies (\ac{Fo} Fo47 n=54 and \ac{Fo} f.sp. \textit{niveum} 110407-3-1-1 n=36) (Figure \ref{fig:CECcount}).

The number of \acp{cec} identified in the \ac{Focub} genome assemblies was comparable to the non-pathogenic \ac{Fo} genome assemblies (Table \ref{tab:CandEffNo}, Figure \ref{fig:CECcount}). Moreover, the \ac{Focub} genome assemblies clustered alongside the non-pathogenic \ac{Fo} genome assemblies based on their \ac{cec} profiles (Figure \ref{fig:MaeiHeatmap}). Within the \ac{Focub4} genome assemblies, a similar number of \acp{cec} were identified, ranging from 42 (B2) to 46 (UK0001) (Table \ref{tab:CandEffNo}). In contrast, the number of \acp{cec} varied slightly more within the \ac{Focub1} genome assemblies, with 34 identified in isolate 60 and 49 in isolate 160527 (Table \ref{tab:CandEffNo}).

While \ac{Focub} assemblies generally clustered together based on their \ac{cec} profiles, variations between races and isolates were observed. Notably, the assembly for \ac{Focub1} isolate 60 was the only \ac{Focub} isolate assembly to cluster separately from the other \ac{Focub} assemblies based on its \ac{cec} profile (Figure \ref{fig:MaeiHeatmap}). Despite being reported as an \ac{r1} isolate by \textcite{Yun2019}, the \ac{cec} profile of \ac{Focub} 60 is more akin to that of the \ac{Fo} endophyte, Fo47, than other \ac{Focub} assemblies. It is of note that \textcite{Yun2019} did not satisfy Koch's postulates for \ac{Focub1} 60, instead classifying it as \ac{Focub1} based on \ac{pcr} analysis. Further, no \acp{sixg} were identified in the \ac{Focub1} 60 genome assembly in Chapter 2 (see section \ref{sec:chap2SixGene}). 

The remaining \ac{Focub} genome assemblies are grouped within a single \ac{Focub} \ac{cec} profile cluster (Figure \ref{fig:MaeiHeatmap}), further dividing into three distinct \ac{Focub} \ac{cec} groups. The first group encompasses two \ac{Focub1} genome assemblies (N2 and 160527) as well as VPR144079 and VPR144084. Phylogenetic analysis based on the \ac{tef} gene revealed a polyphyletic evolutionary history for \ac{Focub}, which has also been observed by \textcite{Odonnell1998, Groenewald2006, Fourie2009, Maryani2019, Mostert2022}. According to the \ac{tef} phylogeny, \ac{Focub1} isolate 160527, and \ac{Focub} VPR144079, and VPR144084 form a monophyletic \ac{tef} group alongside \ac{tnau} isolate, S6 (Group 3, Figure \ref{fig:TEF1aPhyloaMaei}). Conversely the \ac{Focub1} isolates N2 and 60, occupy a separate \ac{tef} group (Group 5, Figure \ref{fig:TEF1aPhyloaMaei}), displaying a closer relationship to other \ac{Fo} \acp{fsp} than the \ac{Focub1} isolate 160527, and VPR144079 and VPR144084. However, the \ac{Focub} \ac{cec} profile groups do not reflect the \ac{tef} phylogeny (Figures  \ref{fig:TEF1aPhyloaMaei} and \ref{fig:MaeiHeatmap}). Although VPR144079 and VPR144084 exhibit highly similar \ac{cec} profiles, the \ac{cec} profile of \ac{Focub1} isolate 160527 more closely resembles that of \ac{Focub1} isolate N2 than that of \ac{Focub} isolates VPR144079 and VPR144084 (Figure \ref{fig:MaeiHeatmap-banana}).

The second \ac{Focub} \ac{cec} profile group contains all \ac{Focub4} genome assemblies and corresponds to the \ac{tef} Group 2 (Figures  \ref{fig:TEF1aPhyloaMaei} and \ref{fig:MaeiHeatmap}). A collective set of 52 \acp{cec} were identified across the \ac{Focub4} genome assemblies. The third  \ac{Focub} \ac{cec} profile group comprises VPR144081, VPR144082, and VPR144083. Although these isolates are from the same \ac{tef} group as \ac{Focub4} (\ac{tef} Group 2, Figure \ref{fig:TEF1aPhyloaMaei}), they exhibit a slightly different \ac{cec} profile (Figure \ref{fig:MaeiHeatmap-banana}). 

\begin{sidewaysfigure}[hp!]
    \centering
    \includegraphics[width=22cm]{Figures/CecDistribinFspOfInterest.png}
    \captionsetup{width=20cm}
    \caption[\Acl{cec} count across \acl{Fs}, \ac{tnau} isolate SY-2, \acl{Fo} non-pathogens, and the \ac{Fo} \acp{fsp} \textit{apii, coriandrii, cubense} and \textit{lactucae}.]{\textbf{\Acf{cec} count across \acl{Fs}, \ac{tnau} isolate SY-2, \acf{Fo} non-pathogens, and the \ac{Fo} \acp{fsp} \textit{apii, coriandrii, cubense} and \textit{lactucae}.} Point size indicates genome assembly size and colour denotes race. Genome assemblies for which race has not been assigned are shown in black. np = non-pathogen.}
    \label{fig:CECcount}
\end{sidewaysfigure}


\begin{sidewaysfigure}[htp!]
    \centering
    \includegraphics[width=\textwidth]{Figures/HeatmapAndPhylo_BananaPathOnly.png}
    \captionsetup{width=24cm}
    \caption[CEC profiles of isolates  causing \acl{fwb} isolates ordered by maximum likelihood phylogeny.]{\textbf{CEC profiles of isolates  causing \acl{fwb} isolates ordered by \ac{tef} maximum likelihood phylogeny.} \ac{FFSC} shown in pink, \ac{FOSC} shown in yellow. The \ac{Fo} non-pathogen, Fo47 was also included. Branch length show expected number of substitutions per site. White boxes indicate bootstrap values from 1000 bootstrap replicates. IQ-TREE2 (v2.2.0.3) best model of substitution; TNe+G4. \Acp{cec} were determined using CD-HIT (v4.8.1) at 80\% identity, following \ac{ce} identification using TBLASTN, (cut-off 1e-6 65\% identity and coverage threshold) and filtering using SignalP (v5.0b) and EffectorP (v2.1.0). np indicates non-pathogen.}
    \label{fig:MaeiHeatmap-banana}
\end{sidewaysfigure}

\subsection{Conserved \aclp{cec} in \textit{Fusarium} isolates pathogenic towards banana.} 

As the \textit{Fusarium} banana pathogens shared multiple \acp{cec}, the distribution of \acp{cec} between \textit{Fusarium} banana pathogens was counted (Figure \ref{fig:UpSetCECofBanana}). None of the \textit{Fusarium} banana pathogens were found to possess \acp{cec} exclusive to them and absent in the \ac{Fo} endophyte, Fo47. Nine core \acp{cec} were identified in SY-2, FS66, and NRRL\_66326, two of which were unique to SY-2,  FS66 and NRRL\_66326, and four were conserved among all \textit{Fusarium} banana pathogens and the \ac{Fo} endophyte (Fo47) (Figure \ref{fig:UpSetCECofBanana}). Moreover, a total of 38 \acp{cec} were shared between the \ac{Fo} endophyte (Fo47) and at least one \ac{Focub} genome assembly (Figure \ref{fig:UpSetCECofBanana}). Only one \ac{cec} was identified in all \ac{Focub} genome assemblies but not the \ac{Fo} endophyte (Fo47) (CEC\_286) (Figures \ref{fig:UpSetCECofBanana} and \ref{fig:MaeiHeatmap-banana}). The \ac{Fo} endophyte (Fo47) contained an additional set of 16 \acp{cec} not identified in any of the \textit{Fusarium} banana pathogen genome assemblies.

Two \acp{cec} were identified in all \ac{Focub} genome assemblies except for  \ac{Focub1} 60 and the \ac{Fo} endophyte (Fo47) (Figure \ref{fig:UpSetCECofBanana}). Additionally, two \acp{cec} were shared among all \ac{Focub4} genome assemblies but not the \ac{Focub1} or \ac{Focub} genome assemblies without assigned races (isolate codes starting VPR1). Seven \acp{cec} were shared between the \ac{Focub} genome assemblies without assigned race from \ac{tef} Group 2, and six \acp{cec} were shared between the \ac{Focub} genome assemblies without race assigned from \ac{tef} Group 3 (Figure \ref{fig:TEF1aPhyloaMaei}). Intriguingly, one \ac{cec}, CEC\_129, was identified in all the isolates virulent towards Cavendish banana (all \ac{Focub4} isolates, \ac{Fs} isolate FS66, and \ac{tnau} isolate S6),  but not the other banana pathogens (\ac{Focub1}, \ac{Focub} genome assemblies without assigned races (isolate codes starting VPR1), and \ac{Fs} isolate NRRL\_66326), suggesting CEC\_129 may play a role in virulence towards Cavendish banana (Figures \ref{fig:UpSetCECofBanana} and \ref{fig:MaeiHeatmap-banana}). 

    
\begin{sidewaysfigure}[hp!]
    \centering
    \includegraphics[width=\textwidth]{Figures/UpSetCECofBanana.png}
    \captionsetup{width=20cm}
    \caption[Upset plot of \ac{cec} distribution between \textit{Fusarium} banana pathogens.]{\textbf{Upset plot showing the distribution of \acp{cec} sets identified using the \ac{maei} pipeline in \textit{Fusarium} banana pathogens.}
    \ac{Focub4} is shown in dark yellow,  \ac{Focub1} is shown in light yellow. \ac{Focub} isolates that do not have a designed race are shown in grey, and the \ac{Fo} endophyte, Fo47, is shown in brown. \acl{Fs} isolates and SY-2 are shown in pink. \Acp{cec} contain at least two \aclp{ce}. Each vertical line represents a set with intersections indicated by circles. The top bar plot records intersection size (number of shared CECs). The bar plot on the right-hand side counts the size of each set (total number of CECs identified in the assembly). The plot was generated using the ggupset (v0.3.0) \parencite{ggupset} package in R (v4.3.1) \parencite{R}.}
    \label{fig:UpSetCECofBanana}
\end{sidewaysfigure}



\subsection{Variation in \acl{Fola} effector compliment.}
\label{FolaMaei}

Although the \ac{Fo} genome assemblies typically clustered into \ac{cec} profile groups by \acp{fsp} then race, assemblies from the same race did not consistently have identical \ac{cec} profiles, as the \ac{Fola} assemblies included in this study demonstrate. All \ac{Fola} isolates formed a monophyletic clade based on the \ac{tef} phylogeny with no obvious race grouping (\ac{tef} Group 5, Figure \ref{fig:TEF1aPhyloaMaei}). The \ac{Fola} genome assemblies also clustered together based on \ac{cec} profile, separating into two race-specific groups. The \ac{Fola} \ac{r1} \ac{cec} profile group had a mean \ac{cec} count of 80 (range: 77 to 87). \ac{Fola} \ac{r1} primarily causes disease in field-grown lettuce varieties in Asia, USA, Southern Europe, and South America. The \ac{Fola} \ac{r4} \ac{cec} profile group had a mean \ac{cec} count of 87  (range: 84 to 90). \ac{Fola} \ac{r4} affects greenhouse-grown (protected) lettuce in the Netherlands and Belgium \parencite{Mestdagh2023}. 

A total of 119 \acp{cec} were identified among the \ac{Fola} genome assemblies, 56 of which were shared between all the \ac{Fola} isolates (Figure \ref{fig:UpSetCECofFola}). Of the 119 \acp{cec}, 41 were shared with the \ac{Fo} endophyte (Fo47). A total of 24 \acp{cec} were identified in all the \ac{Fola} but not the \ac{Fo} endophyte (Fo47), and a further set of 18 \acp{cec} were only identified in \ac{Fola} \ac{r4} genome assemblies (AJ516, AJ592, and AJ705), whereas 15 were only identified in \ac{Fola} \ac{r1} genome assemblies (AJ520, AJ5865, and AJ718) (Figure \ref{fig:UpSetCECofFola}). 

The genome assemblies for \ac{Fola} \ac{r4} isolates AJ592 and AJ705 displayed very similar \ac{cec} profiles, sharing 85 \acp{cec}. \ac{Fola} \ac{r4}  AJ592 contained an additional two \acp{cec} compared to \ac{Fola} \ac{r4}  AJ705 (CEC\_241 and CEC\_47) (Figures \ref{fig:UpSetCECofFola} and \ref{fig:MaeiHeatmap-lettuce}).  Further, the genome assemblies for \ac{Fola} \ac{r4} isolates  AJ592 and AJ705 shared four \acp{cec} that were not identified in \ac{Fola} \ac{r4}  AJ516 or \ac{Fola} \ac{r1}. The \ac{cec} profile of \ac{Fola} \ac{r4}  AJ516 is less like \ac{Fola} \ac{r4} AJ592 and AJ705 (Figure \ref{fig:MaeiHeatmap-lettuce}), containing an additional nine \acp{cec} not identified in \ac{Fola} \ac{r4} AJ592 and AJ705 (Figure \ref{fig:UpSetCECofFola}). 

% Of the 87 \acp{cec} identified in \ac{Fola} \ac{r1}, 76 are core \acp{cec} (Figure \ref{fig:UpSetCECofFola}). \ac{Fola} \ac{r1} AJ520 shows an expanded \ac{cec} profile compared to AJ865 and AJ718, with with all 87 \acp{cec} identified in \ac{Fola} \ac{r1} identified (Figure \ref{fig:UpSetCECofFola}, Table \ref{tab:CandEffNo}).

\begin{sidewaysfigure}[hp!]
    \centering
    \includegraphics[width=\textwidth]{Figures/UpSetCECofFola.png}
    \captionsetup{width = 22cm}
    \caption[Upset plot of \ac{cec} distribution between \ac{Fola}.]{\textbf{Upset plot showing the distribution of \acp{cec} sets identified using the \ac{maei} pipeline in \ac{Fola}.}
    \ac{Fola} \ac{r4} is shown in dark green,  \ac{Fola} \ac{r1} is shown in light green. The \ac{Fo} endophyte, Fo47, is shown in brown. Each vertical line represents a set with intersections indicated by circles. The top bar plot records intersection size (number of shared CECs). The bar plot on the right-hand side counts the size of each set (total number of CECs identified in the assembly). The plot was generated using the ggupset (v0.3.0) \parencite{ggupset} package in R (v4.3.1) \parencite{R}.}
    \label{fig:UpSetCECofFola}
\end{sidewaysfigure}


\begin{sidewaysfigure}[htp!]
    \centering
    \includegraphics[width=\textwidth]{Figures/HeatmapAndPhylo_LactucaeOnly.png}
    \captionsetup{width=24cm}
    \caption[CEC profiles of \ac{Fola} isolates ordered by \ac{tef} maximum likelihood phylogeny.]{\textbf{CEC profiles of \ac{Fola} isolates ordered by \ac{tef} maximum likelihood phylogeny.}  Branch length show expected number of substitutions per site. White boxes indicate bootstrap values from 1000 bootstrap replicates. IQ-TREE2 (v2.2.0.3) best model of substitution; TNe+G4. \Acp{cec} were determined using CD-HIT (v4.8.1) at 80\% identity, following \ac{ce} identification using TBLASTN, (cut-off 1e\textsuperscript{-6} 65\% identity and coverage threshold) and filtering using SignalP (v5.0b) and EffectorP (v2.1.0). \ac{Fo} isolate Fo47 was included for \ac{cec} profile comparison against non-pathogen. np indicates non-pathogen.}
    \label{fig:MaeiHeatmap-lettuce}
\end{sidewaysfigure}

\subsection{Diverse \textit{SIX8} gene phylogenies reveal variability within \acl{Fola} \acl{r4} isolates}

The differences observed in \ac{cec} profile among isolates of the same race, (e.g. \ac{Fola} \ac{r4} AJ516 vs AJ592 and AJ705), may be an artefact of genome assembly, but may also be due to differences in \ac{ce} complement of \ac{Fo} \ac{fsp} strains from the same race. This was explored in  \ac{Fola} races, where we observed sequence variation and differences in copy number of \textit{SIX8} (only found in \ac{Fola} \ac{r4}), \textit{SIX9}, and \textit{SIX14} homologues. In each \ac{Fola} \ac{r4} genome assembly, a single copy of \textit{SIX8} was identified. \textit{SIX8} in AJ592 and AJ705 were identical, but when aligned to \textit{SIX8} from AJ516, 16 single base substitutions and one indel (position 528 to 535) were identified. The two distinct variants of the \textit{SIX8} sequence in \ac{Fola} \ac{r4} were consistently detected among 39 isolates collected from Italy, Ireland, the Netherlands, Spain, and the UK. Among these isolates, 23 exhibited an identical sequence to the \ac{Fola} \ac{r4} isolate AJ516, while six isolates displayed the sequence shared by AJ592 and AJ705 (data not presented). The \textit{SIX8} sequences from \ac{Fola} were  most closely related to the \ac{Fo} isolate from rocket (AJ174), \ac{Fo} \ac{fsp} \textit{conglutinans} (Fo5176), and \ac{Fo} \ac{fsp} \textit{matthiolae} (AJ260) (Figure \ref{fig:lactucae-six8}).

\begin{figure}[htp!]
    \centering
    \includegraphics[width=13cm]{Figures/lactucaeSIX8tree.png}
    \captionsetup{width=\textwidth}
    \caption[Maximum likelihood phylogeny from alignment of \textit{SIX8} sequences from \acl{Fo} reveals differences in \acl{Fola} \acl{r4} \textit{SIX8}.]{\textbf{Maximum likelihood phylogeny from alignment of \textit{SIX8} sequences from \acf{Fo} reveals differences in \acf{Fola} \acf{r4} \textit{SIX8}.}  Branch length show expected number of substitutions per site. White boxes indicate bootstrap values from 1000 bootstrap replicates. IQ-TREE2 (v2.2.0.3) best model of substitution; K2P+G4. The tree is rooted with the reference sequence FJ755837.1 from \ac{Foly} \textit{SIX8}.}
    \label{fig:lactucae-six8}
\end{figure}

\textit{SIX9} copy number varied between \ac{Fola} races and isolates, with two copies identified in \ac{Fola} \ac{r1}, four copies identified in \ac{Fola} \ac{r4} AJ516, and five copies identified in \ac{Fola} \ac{r4} isolates AJ592 and AJ705. Though there was variation between the two copies of \textit{SIX9} in the \ac{Fola} \ac{r1} isolates, which were in separate clades, there was no variation in these copies between isolates (Group 1 and Group 3, Figure \ref{fig:lactucae-six9}). One of the \textit{SIX9} copies appears to have been duplicated in \ac{Fola} \ac{r4}, with two copies of \textit{SIX9} in each isolate that are identical to the \ac{Fola} \ac{r1} copy (Group 3, Figure \ref{fig:lactucae-six9}). Additional copies of \textit{SIX9}, found in all the \ac{Fola} \ac{r4} isolates (but absent in \ac{Fola} \ac{r1}), exhibited similarity to \textit{SIX9} copies identified in \ac{Fo} from rocket (AJ174), \ac{Fo} \acp{fsp} \textit{coriandrii} (3-2), \textit{conglutinans} (Fo5176), and \textit{matthiolae} (AJ260), mirroring the pattern observed with \textit{SIX8} (Group 2, Figure \ref{fig:lactucae-six9}). Additionally, copies of \textit{SIX9} identified in \ac{Fola} \ac{r4} exhibited similarity to a copy of \textit{SIX9} in \ac{Foci} (Group 4, Figure \ref{fig:lactucae-six9}). A further \textit{SIX9} sequence variant was identified in \ac{Fola} \ac{r4} isolates AJ592 and AJ705, but not AJ516 (Group 1, Figure \ref{fig:lactucae-six9}). 

\begin{figure}[htp!]
    \centering
    \includegraphics[width=12cm]{Figures/lactucaeSIX9tree.png}
    \captionsetup{width=\textwidth}
    \caption[Maximum likelihood phylogeny from alignment of \textit{SIX8} sequences from \acl{Fo} reveals differences in \acl{Fola} \textit{SIX9}.]{\textbf{Maximum likelihood phylogeny from alignment of \textit{SIX9} sequences from \acf{Fo} reveals differences in \acf{Fola} \textit{SIX9}.}  Branch length show expected number of substitutions per site. White boxes indicate bootstrap values from 1000 bootstrap replicates. IQ-TREE2 (v2.2.0.3) best model of substitution; TPM3+G4. The tree is rooted with the reference sequence \ac{Foly} \textit{SIX9} reference KC701447.1.1.}
    \label{fig:lactucae-six9}
\end{figure}

Each of the \ac{Fola} \ac{r4} isolates possessed one identical copy of \textit{SIX14}, two identical copies of the same \textit{SIX14} was identified in each \ac{Fola} \ac{r1} isolate (Group 1, Figure \ref{fig:lactucae-six14}). The \ac{Fola} \ac{r1} isolates possessed a third copy of \textit{SIX14} (Group 2, Figure \ref{fig:lactucae-six14}). The sequence variants of \textit{SIX14} across all \ac{Fola} isolates exhibited closer resemblance to each other than to homologues of \textit{SIX14} in other \ac{Fo} \acp{fsp}.

\begin{figure}[htp!]
    \centering
    \includegraphics[width=12cm]{Figures/lactucaeSIX14tree.png}
    \captionsetup{width=\textwidth}
    \caption[Maximum likelihood phylogeny from alignment of \textit{SIX14} sequences from \acl{Fo} reveals differences in copy number of \textit{SIX14} between races of \acl{Fola}.]{\textbf{Maximum likelihood phylogeny from alignment of \textit{SIX14} sequences from \acf{Fo} reveals differences in copy number of \textit{SIX14} between races\acf{Fola}.}  Branch length show expected number of substitutions per site. White boxes indicate bootstrap values from 1000 bootstrap replicates. IQ-TREE2 (v2.2.0.3) best model of substitution; K2P+G4. The tree was plotted using the R \parencite{R} package, ggtree \parencite{ggtree}. The tree is rooted with the reference sequence \ac{Foly} \textit{SIX14} reference KC701452.1.}
    \label{fig:lactucae-six14}
\end{figure}

\subsubsection{Expression of \aclp{ce} identified using the \ac{maei} pipeline in \acl{Fola}.}

To assess whether \acp{ce} identified using the \ac{maei} pipeline were expressed during infection, lettuce seedlings (\textit{Lactuca sativa} cv. Kordaat) were inoculated with \ac{Fola} \ac{r4} (AJ516 and AJ705) and \ac{r1} (AJ520), and RNA extracted (96 \ac{hpi}) and sequenced. Small ($ \leq1$ kb), up-regulated genes, encoding proteins predicted to be secreted \textit{in planta}, were classified as putative effectors. This identified a total of 14 putative effectors among all  \ac{Fola} \ac{r1} and \ac{r4} isolates, seven of which  were identified separately using the \ac{maei} pipeline (CEC\_59, CEC\_6, CEC\_132, CEC\_107, CEC\_64, CEC\_269, CEC\_88). It also  identified a further eight putative effectors specific to \ac{Fola} \ac{r4} isolates, and three specific to \ac{Fola} \ac{r1}. Of the eight putative effectors specific to \ac{Fola} \ac{r4}, three were found separately using the \ac{maei} pipeline (CEC\_27, CEC\_77, CEC\_190). Interestingly, CEC\_190 (orthogroup OG0017138) was a \ac{Fola} \ac{r4} specific \ac{cec} (Figure \ref{fig:MaeiHeatmap-lettuce}). One of the three \ac{Fola} \ac{r1} specific putative effectors identified as part of the RNAseq was also identified using the \ac{maei} pipeline, CEC\_155. 

\subsection{Distribution of \aclp{ce} from \ac{tnau} genome assemblies in \textit{Fusarium} pathogenic towards banana \ac{ce} sets}
\label{sec:Chap3RNASeq}

A search was conducted for reciprocal best hits of \acp{ce} from the \ac{tnau} genome assemblies, S6 (\acp{ce} = 333), S16  (\acp{ce} = 289), S32 (\acp{ce} = 314), and SY-2 (\acp{ce} = 289) (see section \ref{tnauCEs}). This search was performed within the \ac{ce} sets identified using the in the \ac{maei} pipeline, from genome assemblies of \textit{Fusarium} strains pathogenic to banana, \ac{Focub} UK0001, (\acp{ce} = 127), 160527 (\acp{ce} = 95), VPR144083 (\acp{ce} = 50), VPR144084 (\acp{ce} = 104), and \ac{Fs} isolate FS66 (\acp{ce} = 12). Reciprocal best hits were determined utilising MMseqs2 easy-rbh (v13.45111), with a minimum threshold of 20\% identity, 45\% alignment length, and a bitscore of at least 30. 

Among the 333 \acp{ce} in the \ac{tnau} S6 genome assembly, 17 had reciprocal best hits in the \ac{ce} sets from \textit{Fusarium} pathogenic towards banana. CE\_c89g74, a potential pyruvate decarboxylase, was found in all \acp{ce} sets from \textit{Fusarium}  pathogenic towards banana. Additionally, six of the S6 \acp{ce} exhibited reciprocal best hits in all \ac{Focub} \ac{ce} sets. Conversely, one \ac{ce} displayed a reciprocal best hit solely in the FS66 \ac{ce} set (Figure \ref{fig:RBHupsets}a). 

Fewer \acp{ce} identified in the \ac{tnau} genome assemblies, S16, S32, and SY-2, exhibited reciprocal best hits ($\geq20\%$ identity, $\geq45\%$ alignment length, $\geq30$ bit score) when compared to S6, with nine, six, and nine identified, respectively (Figure: \ref{fig:RBHupsets}). Two \acp{ce} from S16, S32, and SY-2 were identified in the \ac{Fs} FS66 \ac{ce} set, in contrast to only one \ac{ce} from S6. One \ac{ce} (CE\_c05g1317) from S16 and one \ac{ce} (CE\_c12g253) from SY-2 displayed reciprocal best-hits in all \ac{ce} sets derived from \textit{Fusarium} pathogenic towards banana (Figure \ref{fig:RBHupsets}b,d). Notably, CE\_c12g253 and CE\_c05g1317 are identical, and share homology with a cyanate hydratase found in other \textit{Fusarium} species (web-based BLASTP search of \ac{ncbi} nr database; \href{https://blast.ncbi.nlm.nih.gov/Blast.cgi}{https://blast.ncbi.nlm.nih.gov/Bla\-st.cgi}).
 
\begin{figure}[hp!]
    \centering
    \includegraphics[width=\textwidth]{Figures/UpSetPlots.png}
    \caption[Distribution of reciprocal best-hits of \ac{tnau} \acp{ce}  in \ac{Focub} and \ac{Fs} \ac{ce} sets identified using the in the \ac{maei} pipeline.]{\textbf{Distribution of reciprocal best-hits of \ac{tnau} \acp{ce} in \ac{Focub} and \ac{Fs} \ac{ce} sets identified using the in the \ac{maei} pipeline.} \textbf{a)} Distribution of reciprocal best-hits of S6 \acp{ce}, \textbf{b)} S16 \acp{ce}, \textbf{c)} S32 \acp{ce} and \textbf{d)} SY-2 \acp{ce}. Each vertical line represents a set with intersections indicated by circles. The top bar plot records intersection size (number of shared CEs). These plots were generated using the ggupset (v0.3.0) \parencite{ggupset} package in R (v4.3.1) \parencite{R}.}
    \label{fig:RBHupsets}
\end{figure}

\clearpage
\section{Discussion and conclusion}

Using the \ac{maei} pipeline, a total set of 8,678 \acp{ce} were identified across 42 \textit{Fusarium} genome assemblies. These 8,678 \acp{ce} were clustered at 65\% identity, resulting in 325 \acfp{cec}. The number of \acp{mimp}, \acp{ce}, and \acp{cec} varied among \acp{fsp} and genomes within \acp{fsp}. Few \acp{mimp}, \acp{ce}, and \acp{cec} were identified in non-\ac{Fo} genomes (\acl{Fg} and \acl{Fs}), underscoring the specificity of the \ac{maei} pipeline approach. Moreover, \ac{Fo} non-pathogens generally exhibited a lower \ac{ce} count compared to \ac{Fo} pathogens, with \ac{Focub} being a notable exception.

Relationships were observed, not only between \ac{mimp} and \ac{ce} content, and \textit{Fusarium} lifestyle,  but also \ac{mimp} and \ac{ce} content, and genome assembly size. Both \ac{Fo} non-pathogens, Fo47 and 110407-3-1-1, had genome assembly sizes of 49.7 Mb and 50.4 Mb, containing 81 \acp{ce} and 39 \acp{ce}, respectively. In contrast, the \ac{Fola} \ac{r4} isolate AJ516 genome assembly and the \ac{Foci} isolate AJ615 genome assembly had larger genome sizes of 68.8 Mb and 69.3 Mb, respectively, containing 548 \acp{ce} and 603 \acp{ce}. However, it remains unclear whether the larger \ac{mimp} and \ac{ce} count is due to a genuine expansion in \ac{mimp} and \ac{ce} content in pathogen genomes or if a greater number of false positive \acp{mimp} and \acp{ce} are identified due to the larger genome assembly size. The \ac{Focub} genomes present an opportunity to investigate this further, although no definitive conclusions are drawn.

The \ac{Focub} genomes have an average assembly size of 48.26 Mb, compared to the averages of 64.47 Mb for \ac{Fola}, 65.30 Mb for \ac{Foa}, and 67.35 Mb for \ac{Foci}. This investigation identified fewer \acp{mimp}, \acp{ce}, and consequently \acp{cec} in all \ac{Focub} genomes compared to other \ac{Fo} pathogens, including \ac{Fola}, \ac{Foa}, and \ac{Foci} (see Table \ref{tab:CandEffNo} and Figures \ref{fig:MaeiStats} and \ref{fig:CECcount}), consistent with findings reported by \textcite{Dam2016, FoEC2}. A recent preprint by \textcite{Ma2023} found that the \ac{Focub4} isolates included in their study did not contain a full \acf{ac}; the study identified accessory sequences located at the end of \acfp{cc}. Collaborators at \ac{niab}\footnote{Conducted as part of the \acl{Fola} \ac{ce} study, which included the \ac{maei} pipeline. A publication is under review: \textcite{FolaManuscript}.} identified core and accessory genomes in \ac{Fo} assemblies (personal communication). They found that all \ac{Fo} isolates' core genome comprised approximately 45-55 Mb, but that accessory genome varied from 23.5 Mb in \ac{Fo} \ac{fsp} \textit{lini} 39\_C0058 to 1.3 Mb in \ac{Focub1} 160527. 

Isolates with larger accessory genomes, and consequently larger overall genome sizes, exhibited higher \ac{mimp} and \ac{ce} content compared to isolates with smaller accessory genomes and smaller overall genomes. For instance, the \ac{Fola} \ac{r4} genome assemblies for isolates AJ516, AJ592, and AJ705 contained 47.5 Mb, 46.2 Mb, and 46.5 Mb of core genome contigs, and 20.9 Mb, 18.2 Mb, and 17.7 Mb of accessory genome contigs and 548, 482, and 490 \acp{ce}, respectively. The core genome contigs of \ac{Foci} 3-2, \ac{Foa} 207.A, and \ac{Fo} \ac{fsp} \textit{lini} 39\_C0058 were 46.3 Mb, 46.1 Mb, and 45.8 Mb in length, respectively, and their accessory genome contigs were 15.5 Mb, 18.3 Mb, and 23.5 Mb, respectively, while \acp{ce} totalled 315, 388, 237, respectively. In contrast, the non-pathogen \ac{Fo} Fo47 contained 46.1 Mb of core genome contigs, 4.3 Mb of accessory genome contigs, and 81 \acp{ce}. The non-pathogen \ac{Fo} \ac{fsp} \textit{niveum} 110407-3-1-1 genome assembly contained 46.5 Mb of core genome contigs, 3.1 Mb of accessory genome contigs, and 39 \acp{ce}. For \ac{Focub} isolates UK0001 (\ac{tr4}) and 16052 (\ac{r1}), collaborators identified core genomes of 47.1 Mb and 49.9 Mb and accessory genomes of 1.4 Mb and 1.3 Mb, respectively; while we identified 127 \acp{ce} in the \ac{Focub4} UK0001 isolate, and 95 \acp{ce} in the \ac{Focub1} 160527. The larger accessory genome and increased \ac{mimp} and \ac{ce} content suggest that the greater number of \acp{mimp} and \acp{ce} identified through the \ac{maei} pipeline is reasonable.

\bigskip
\noindent
Through hierarchical clustering of the presence/absence pattern of the 325 \acp{cec}, \textit{Fusarium} genomes primarily clustered according to \ac{fsp} (host-specificity), and then race. This clustering pattern enhances our confidence in the \acp{ce} identified using the \ac{maei} pipeline. Notable exceptions to the general clustering pattern (by host-specificity) were observed for the \ac{Foa} and \ac{Foci} genomes. \ac{Foa} \ac{r3} and \ac{r4} clustered as a separate group from \ac{Foa} \ac{r2} and \ac{Foci} isolate 3-2 clustered separately from \ac{Foci} isolate AJ615. In fact, \ac{Foci} isolate 3-2 clusters more closely with \ac{Foa} \ac{r3} and \ac{r4} than \ac{Foci} isolate AJ615. Likewise, \ac{Foci} isolate AJ615 clusters more closely to \ac{Foa} \ac{r2} (Figure \ref{fig:MaeiHeatmap}). 

As \textcite{Henry2020} demonstrate, \ac{Foa} \ac{r3}, \ac{r4}, and \ac{Foci} isolates 3-2 and GL306 have similar genomes and form a monophyletic group, which is supported by our \ac{tef} phylogeny (Figure \ref{fig:TEF1aPhyloaMaei}). \ac{Foa} \ac{r4} can also cause disease in coriander (\textit{Coriandrum sativum}), the host of \ac{Foci} \parencite{Epstein2022}. \textcite{Henry2020, Epstein2022} show that \ac{Foa} \ac{r2} is from a different \ac{FOSC} lineage than \ac{r3} and \ac{r4}, and report that \ac{Foa} \ac{r2} and \ac{r4} did not form conidial anastomosis tubes, a means by which \acp{ac} can be horizontally transferred between isolates and a mark of isolate compatibility. \textcite{Epstein2022}, posit that, although \ac{Foa} \ac{r2}, \ac{r3}, and \ac{r4} share the same host species (\textit{Apium graveolens}), the recently described \ac{Foa} \ac{r4} did not evolve from \ac{Foa} \ac{r2} and did not receive a portion of \ac{ac} from this extant pathogen of celery (\textit{Apium graveolens}). Further, \textcite{Epstein2022} show that \ac{Foci} isolate 3-2 can form conidial anastomosis tubes with \ac{Foa} \ac{r4} isolates, and \textcite{Henry2020} demonstrate a close phylogenetic relationship and many genomic similarities between \ac{Foa} \ac{r3}, \ac{r4}, and \ac{Foci} 3-2. 

The observed patterns suggest that the host specificity within \ac{Foa} \ac{r2}, \ac{r3}, and \ac{r4} has likely arisen through distinct mechanisms involving different sets of \acp{ce}. Hierarchical clustering of genomes based on \ac{cec} profiles provides further evidence of this divergence (Figure \ref{fig:MaeiHeatmap}). Moreover, the resemblance between \ac{r4} and \ac{Foci} 3-2, along with the shared host range (\textit{Coriandrum sativum}), is underscored by our hierarchical clustering analysis. The inclusion of \ac{Foci} AJ516, a strain not previously examined by \textcite{Henry2020, Epstein2022}, adds complexity to our understanding. Notably, \ac{Foci} AJ615 exhibits a larger accessory genome and a divergent \ac{cec} profile compared to \ac{Foci} 3-2, and \ac{Foci} AJ615 originates from Portugal, where as the \ac{Foci} isolates studied by \textcite{Henry2020, Epstein2022} (\ac{Foci} 3-2 and GL306) originate from the USA. 

As well as variation among \acp{fsp}, the \ac{maei} pipeline can identify variation within \ac{fsp}. The \ac{Fola} \ac{r4} isolate AJ516 \ac{cec} profile varied from that of the other \ac{Fola} \ac{r4} isolates (AJ592 and AJ705). Analysis of known effector content (\textit{SIX8, SIX9}, and \textit{SIX14}) revealed differences in the \textit{SIX8} sequence and \textit{SIX9} copy number between the \ac{Fola} \ac{r4} isolate AJ516 and the \ac{Fola} \ac{r4} isolates AJ592 and AJ705. 

\ac{Fola} was also used to confirm transcription of some \acp{ce}. An independent RNAseq analysis of lettuce (\textit{Lactuca sativa}) seedlings inoculated with \ac{Fola} \ac{r4} (n=2) and \ac{r1} (n=1) isolates was conducted by collaborators at \ac{niab} (personal communication)\footnote{Conducted as part of the \acl{Fola} \ac{ce} study, which included the \ac{maei} pipeline. A publication is under review: \textcite{FolaManuscript}.}. They identified small ($\le1$Kb) secreted, up-regulated genes \textit{in planta} and classified them as putative effectors. Out of their 25 putative effectors, 11 of our \acp{cec} matched in location and genomic sequence, confirming that the \ac{maei} pipeline can identify \acp{ce} that are expressed \textit{in planta}. It is of note that the RNAseq samples were collected at 96 \ac{hpi} (4 \ac{dpi}) (timed to coincide with \textit{SIX8} delivery), and that other \acp{ce} we identified in \ac{Fola} may also be expressed \textit{in planta} but not at this time point, and so were not detected. So far, we have been unable to generate any \textit{Fo} mutants to confirm our \acp{ce} play a role in isolate virulence. 

\textcite{Toruno2016} reviewed the spatial-temporal delivery of pathogen effectors into plant hosts and showed that effectors are deferentially expressed throughout the infection process or in a histologically specific manner. Limited reports for \ac{Fola} have been published, however, \textcite{Li2011} demonstrated that \ac{Focub4} chlamydospores attached to banana roots and produced germination tubes between 3 and 6 \ac{dpi}. \textcite{Li2017} observed that spores and hyphae of both \ac{Focub1} and \ac{tr4} had attached to banana root hairs and root epidermis at 2 \ac{dpi}, and reported that the hyphae of both \ac{Focub1} and \ac{tr4} were found in the vascular tissues of roots by 3 \ac{dpi}. These studies suggest that, for \ac{Focub} at least, key steps of host penetration take place before the 96 \ac{hpi} (4 \ac{dpi}) RNAseq sampling time point. However, it is important to recognise that \textcite{Li2017} report \ac{Focub1} penetrates the root surface of \ac{Focub1} tolerant `Brazil Cavendish`, and the authors later observe (7 \ac{dpi})  \ac{Focub4} in the parenchymal cells of banana root, but not \ac{Focub1}. It may also be reasonable to conclude that effectors required for successful host infection are expressed after the 4 \ac{dpi} time point.  


\bigskip
\noindent
The use of bioinformatics software (e.g. SignalP and EffectorP) as part of the \ac{maei} pipeline design is beneficial, as updates and improvements to these tools can be implemented and will potentially enable the discovery of more, robust \acp{ce}. For instance, since the \ac{maei} pipeline was developed, a new version of EffectorP (v3.0) has been published. The updated version of EffectorP has been trained on a larger dataset than previous versions (EffectorP $\le$ v2.0), and now classifies \acl{ce} into apoplastic or cytoplasmic effectors \parencite{Sperschneider2022}. However, as \textcite{Sperschneider2015c} demonstrate, it is worth validating the efficacy of any updates. \textcite{Sperschneider2015c} assessed three versions of SignalP (v2, v3, and 4) against a panel of known fungal and oomycete effectors and showed that a SignalP 4, was unable to reliably predict the signal peptide of the oomycete Crinkler effectors in the test set. 

As \textcite{LoPresti2015, Sperschneider2015, Sperschneider2015c} explain, filtering secreted proteins (e.g. based on size, cysteine content, the lack of a transmembrane domain, encoded by genes in gene-sparse, repeat-rich genomic regions) may result in genuine effectors being missed. Following this, the \ac{maei} pipeline limits additional filtering steps. However, an amino acid size filter ($\geq30$ aa and $\leq450$ aa) was included for AUGUSTUS (v3.3.3) gene models and \acp{orf}, to reduce the number of gene models and \acp{orf} for SignalP (v5.0b) to process, as this was the most time-consuming stage of the analysis for each \textit{Fusarium} genome. 

Though the \ac{maei} pipeline is designed to identify \acp{ce} in specific \ac{Fo} genomes, we clustered these \acp{ce} at 65\% identity and performed hierarchical clustering of the \textit{Fusarium} genomes based on the \ac{cec} presence/absence. The 65\% identity threshold was somewhat arbitrary but was chosen as, at this threshold, a set of 'core  \acp{cec}' were identified and \textit{Fusarium} genomes separated by \ac{fsp} and race-specific groups, consistent with \textcite{Dam2016,FoEC2}. At a higher percentage identity threshold, little to no \ac{Fo} `core \acp{cec}` were detected, and, although genomes from the same \ac{fsp} still grouped, identifying \ac{fsp} specific clusters was more challenging at lower thresholds. Further, generating \acp{cec} based on the 65\% identity allows for variation in \aclp{ce} within a \acp{cec}. Some \acp{ce} may only share 65\% identity, so differences in the sequence of \acp{ce} within a \ac{cec} may influence \ac{ce}, structure or function, and consequently, pathogen virulence. 

Consistent with our conclusions in the second chapter (see Chapter \ref{Chap2}), the \ac{tnau} SY-2 genome assembly displayed a similar \ac{ce} count and \ac{cec} profile to the \ac{Fs} genome assemblies. The \ac{maei} pipeline results further support the classifications of the \ac{tnau} isolates. The \ac{ce} set from the S6 genome had a greater number of reciprocal best-hits in the \ac{Focub} genomes than in the S16, S32, and SY-2 \ac{ce} sets. However, the reciprocal best-hit data is not a clear indicator of \ac{tnau} classification, and it is important to note that the number of \acp{ce} identified in each genome ranged significantly (\ac{Fs} FS66 = 12, S6 = 333), which will impact the overall distribution of reciprocal best-hits. 

Overall, we have developed an improved \acl{ce} prediction pipeline (\ac{maei}) specific to \ac{Fo}, adhering to the recommendations of \textcite{Sperschneider2015, LoPresti2015, Todd2022}. The \ac{maei} pipeline can identify \acp{ce} in \ac{Fo}, some of these \acp{ce} are expressed \textit{in planta}, and have been identified as \acp{ce} separately from the \ac{maei} analysis. We have shown, in line with other publications (e.g. \textcite{Schmidt2013, Dam2016, FoEC2}), that  \acp{mimp} can be used to predict \acp{ce} in \ac{Fo}, and that different \ac{fsp} of \ac{Fo} have different \acp{ce} repertoires. Expanding upon that, we have investigated variance in \ac{cec} profile identified by the \ac{maei} pipeline among isolates from specific races of \ac{Fo} \acp{fsp}, and shown that there is also variance in known effectors among these isolates from the same race of \ac{Fo} \acp{fsp}. The \ac{maei} pipeline is a valuable new tool in the identification of \acp{ce} in \ac{Fo}, and can be used to study pathogen diversity and identify race-specific \acp{ce} for which targeted molecular diagnostics can be developed. 