
\section{Introduction}

Alongside genomic analysis, we developed and employed other "omics" techniques to enhance our understanding of key biological aspects of \ac{Focub} infection and to support the diagnosis of \ac{fwb}. Omics, is defined by \textcite{Dai2022}, as "probing and analysing large datasets representing the structure and function of an entire biological system at a specific level". Numerous omics approaches, including genomics, transcriptomics, proteomics, metabolomics, and phenomics, have emerged in recent decades and have been instrumental in studying plants and their associated biotic and abiotic stresses. Indeed, \textcite{Backiyarani2022} provide a comprehensive review of omics techniques applied in banana research and their contributions to banana improvement. We are witnessing a transition into the era of "multi-omics", where these techniques and datasets are integrated to uncover novel associations between biological systems at various levels \parencite{Hasin2017}.

The adoption of multi-omics approaches for investigating plant diseases has become increasingly prevalent. For example, \textcite{Crandall2020} illustrate how both metabolomics and genomics have deepened our understanding of the salicylic acid pathway's role in plant defence. \textcite{Zhu2021} employed transcriptomics and metabolomics to explore the effects of different soil treatments on \ac{Fo} \ac{fsp} \textit{capsici} infestation in pepper (\textit{Capsicum annuum}). In our study, we employed phenomic analyses, including multispectral and RGB imaging, and \acf{um} analysis, to elucidate banana responses to various wilting stresses and help develop \ac{fwb} disease diagnostic methods.


Multispectral and RGB imaging studies in banana been established for disease detection and diagnosis. \textcite{Johansen2014} developed an automatic banana identification software to aid in Banana Bunch Top Virus inspection and \textcite{Liao2018} employed hyperspectral images and machine learning to diagnose Banana Streak Virus at earlier stages of infection. \textcite{Ochoa2016} designed a hyperspectral imaging system for disease scanning on banana plants focusing on Black Sigatoka. \acp{uav}, RGB imaging, and Artificial Neural Networks have also been employed in the monitoring of Yellow Sigatoka \parencite{Calou2020}.  

Recently, \acf{rs} disease diagnosis efforts in banana have focused on the identification of \ac{Focub} infection \parencite{Ye2020a, Ye2020b, Selvaraj2019}.  Ye et al (2020a,b) demonstrated that \ac{uav}-based multispectral imagery can be used to diagnose \ac{fwb}. The authors observed statistically significant differences (p > 0.05) between healthy and diseased plants using six different \acp{vi}. Using logistic regression models to describe the relationship between the \acp{vi} in healthy or diseased plants, the authors reported that the red-edge chlorophyll index had the best performance for identifying \ac{fwb}.  

It is clear that spectral reflectance can be used to diagnose \ac{fwb} in banana. What is not clear, however, is what is happening from a biological perspective to cause the differences in spectra observed, and whether the changes in spectra can be distinguished from other biotic and abiotic stresses. For instance, as \ac{Focub} is a vascular pathogen, it is important to ensure that \ac{Focub} infection can be differentiated from drought stress and other vasucular wilt pathogens like \acf{xvm}. 

Metabolomics, another vital "omics" field used in biological sciences, is the comprehensive analysis of the spectrum of metabolites within a biological system \parencite{Klassen2017}. Frequently used to track disease progression and pinpoint potential novel metabolites associated with disease and disease susceptibility; metabolomics has become a useful tool in disease diagnostics. As metabolites mirror downstream expression of the genome, transcriptome, and proteome, they offer an intimate snapshot of an organism's phenotype at a specific stage of infection. Scrutinising metabolic variations in infected, resistant, and healthy individuals can unearth disease, susceptibility, or resistance biomarkers. Furthermore, biomarkers may be used for the early identification of diseases, potentially underpinning a pivotal disease management strategy. 
Metabolomics studies are typically classified into two main categories: targeted and untargeted analyses. Targeted metabolomics (TM) approaches identify a specifically selected set of compounds for analysis \parencite{Allwood2021}.. This approach is usually employed when precise quantitative analysis is necessary, often requiring complex extraction protocols. When employing an \ac{um} approach, researchers aim to identify a wide range of features without predefined targets. This facilitates the discovery of new and diverse compounds, including previously unknown compounds and metabolites. \ac{um} typically employs simpler extraction and detection procedures compared to targeted studies. However, it generates highly intricate data, which comes with an increased challenge of false discoveries. Consequently, \ac{um} demands substantial effort in data analysis and interpretation.

Now emerging as a useful tool in plant pathology, \ac{um} has been employed in the study of infection as well as plant resistance and susceptibility \parencite{Allwood2021}. Notably, \textcite{Garcia2018} harnessed \ac{lcms} to pinpoint biomarkers indicative of \textit{Phythopthora infestans} infection in tomato plants, even in asymptomatic cases. Similarly, \textcite{Sambles2017} and \textcite{Sidda2020} used \ac{um} to identify certain secoiridoid glycosides as discriminatory metabolites signifying susceptibility to ash die-back in both UK and Danish ash trees (\textit{Fraxinus excelsior}).

\textcite{Kasote2020} employed a compartmentalised TM and \ac{um} strategy to differentiate watermelon (\textit{Citrullus vulgaris}) plants exhibiting symptoms of \textit{Fusarium} wilt, caused by \ac{Fon}, from their asymptomatic counterparts, while also distinguishing them from healthy plants of distinct varieties. Using this approach, the authors were able to identify biomarkers associated with the progression of \textit{Fusarium} wilt across watermelon varieties. They showed that the metabolic profiles of \textit{Fusarium} wilt-infected plants exhibit distinct variations depending on their genotype, as well as differences between leaf and stem tissues compared to the root. Phytohormones such as jasmonic acid-isoleucine (JA-Ile) and methyl jasmonate (MeJA) accumulated in resistant varieties, whereas indole-3-acetic acid (IAA) was identified in all resistant lines 16 days after inoculation. The authors suggest that IAA can be used as a potential biomarker of \ac{Fon} infection in watermelon. However, as IAA, as well as the other phytohormones identified,  are commonly associated with immune responses, one has to question how effective they will be as a \ac{Fon} specific biomarker. The varying levels of amino acids (Arg, Asp, Cit, His, Val, and Lys) and the phenolic acid, Phthalic acid (PHA) also offer valuable insights into the interaction between \ac{Fon} and watermelon plants. However, it is imperative to conduct further comprehensive investigations to determine the precise roles of these distinctive metabolites in the development of \textit{Fusarium} wilt and their contribution and specificity to watermelon plant resistance against \ac{Fon}.

Most research into the metabolic composition of bananas primarily focuses on the fruit and its relationship with diet. Only a limited number of studies have delved into the metabolic profiles of bananas affected by \ac{Focub}. In a study conducted by \textcite{Li2013c}, the virulence of various \ac{Focub} isolates was assessed. LC/MS/MS analysis was used to quantify the presence of two mycotoxins commonly associated with \ac{Focub}, beauvericin and fusaric acid, in different parts of banana plants such as roots, fruits, pseudostems, and foliage. The findings of \textcite{Li2013c} revealed a strong correlation between virulence and the accumulation of these toxins. Additionally, they investigated the occurrence of these toxins in field-grown plants displaying symptoms of \ac{Focub} infection and found that, while the toxins were present in the fruit, their levels were too low to pose any significant risk to human or animal health.

To the best of our knowledge, there have been no studies to date that have employed \ac{um} to comprehensively analyse the metabolic profile of \ac{Focub}-infected banana plants. One objective of this thesis is to compare the data generated from metabolomics studies with imaging data to shed light on the biological processes contributing to the observed differences during imaging. This approach also aimed to identify specific markers associated with each stress condition. 

\newpage
\section{Materials and Methods}
\label{sec:Chapter4_MM}

\subsection{Plant maintenance}
\textit{In vitro} ‘Grand Naine’ banana plants (VITROPIC, Saint-Mathieu-de-Tréviers, France) were maintained in 1/2L pots in \~ 400g of compost (Levington Advanced M2 compost, BHGS Ltd, UK), at 25$^{\circ}$C in 12-hour light and 70\% relative humidity. Plants were inoculated 8-12 weeks after arrival. 

\subsection{Fungal and bacterial culture maintenance and inoculation}

\subsubsection{\acl{Focub4}}
A 50 \(\mu\)L droplet of \acl{Focub4} (isolate UK0001, provided by Dr Will Kay, Exeter University) 25\% glycerol stock  was added to 400ml of \acf{pdb} and was maintained in a shaking incubator at \~ 130rpm and 25-28$^{\circ}$C (Max 30$^{\circ}$C). After three days, the liquid culture was filtered through two layers of Miracloth, and the filtered suspension was adjusted to 1 × 10\textsuperscript{6} spores ml\textsuperscript{-1} (see Appendix C\ref{apx:mediaRecipies} for media recipes).

‘Grand Naine’ banana plants were inoculated using the root-drench method from \textcite{Garcia-Bastidas2019}. Briefly, plant roots were wound by cutting with a blade through the soil at a 45° angle. A suspension of 1x10\textsuperscript{6} spores g\textsuperscript{-1} of soil was added to the soil by pouring. Negative controls were inoculated using sterile distilled water. Plants were maintained plants in trays covered with clear plastic bags to prevent pathogen spread. 

\subsubsection{\acl{xvm}}
A 50 \(\mu\)L droplet of \acf{xvm} (isolate 5835, University of Warwick collection) 25\% glycerol stock was added to 10ml of \ac{ypgb} and maintained in a shaking incubator at \~ 130rpm and 25-28$^{\circ}$C (Max 30$^{\circ}$C). After two days, the culture was centrifuged at 3000 \ac{rpm} for 10 minutes and the supernatant was discarded. The pellet was re-suspended in sterile distilled water and diluted to O.D\textsubscript{600} of 0.5. 

A 5ml syringe with a 21-gauge needle was inserted into the pseudostem of the ‘Grand Naine’ banana plants approximately 1cm from the base. 2ml of the 108 CFU ml\textsubscript{-1} \ac{xvm} cell concentration (O.D\textsubscript{600} = 0.5) was slowly introduced into psuedostem. Negative controls were inoculated using sterile distilled water. Plants were maintained plants in trays covered with clear plastic bags to prevent pathogen spread. 

\subsection{\Acl{um} sampling and multispectral image collection}
\subsubsection{Sample collection}
Samples and images from four treatments were collected. Plants were inoculated with \ac{Focub} or \ac{xvm}, exposed to drought stress (no watering from the time of \ac{xvm} inoculation), or mock-inoculated with sterile distilled water. Imaging and sample collection was staggered to ensure that symptom scores were similar between treatments. Images, sample scores, and \ac{um} samples were collected plants from plants inoculated with \ac{xvm}, as well as the drought stress and water inoculated plants, at 7, 10, and 13 \ac{dpi} and from plants inoculated with \ac{Focub4} at 15, 18 and 21 \ac{dpi}. 

\subsubsection{Symptom scoring}
External and internal symptom scores were collected following the methods outlined by \textcite{Garcia-Bastidas2019}. Briefly, external symptoms were visually assessed according to the amount of chlorotic/wilting foliage and scored following a 1–4 scale: 1 =  $0\ge x \leq25\%$, 2 = $25\%\ge x \leq50\%$, 3 = $50\%\ge x \leq75\%$, and 4 = $75\%\ge x \leq100\%$. Internal symptoms were also scored. Plants were cut longitudinally at the rhizome, disease severity was visually assessed and recorded using a 1–6 scale:  1 = No discoloration in the corm, 2 = isolated points $\ge x \leq5\%$, 3 = $5\%\ge x \leq30\%$, 4 = $30\%\ge x \leq50\%$, 5 = $50\%\ge x \leq90\%$, and 6 = $90\%\ge x \leq$ plant totally decayed. 

\subsubsection{Image collection}
Images were captured using a Sony Alpha 7Rii modified camera body with 10 band lens for a Full-Frame sensor (405-850nm) (Agrowing Ltd., Israel). Plants were imaged individually. Images were captured at 2m above the canopy using a custom imaging system. Images were aligned and analysed using the software Agrowing Basic (v.1.2.) (Agrowing Ltd., Israel). 

\subsubsection{\Acl{um} sample collection}
For \ac{um} analysis, three leaves were collected per plant, with three plants sampled from each treatment group at each time point. A 75mm by 25mm section of lamina on either side of the midrib was excised from the base of each leaf, snap-frozen in liquid N\textsubscript{2}, lyophilised, and ground to a powder. The powdered lamina were then homogenised to generate a single  sample for each plant. 

%\subsection{X-ray computed tomography image collection}
%We scanned both a single leaf sample and a section of pseudostem from each sample (2 Control Plants; 4 Xanthomonas Plants; 4 FOC Plants) on our Tescan Unitom XL system. The settings used can be found in the table below. All scans were reconstructed using the standard FDK algorithm. The X-Ray Computed Tomography (XCT) data used in this report was acquired using the Free-at-Point-of-Access scheme at the National Facility for X-Ray Computed Tomography (NXCT) and carried out at the Centre for Imaging, Metrology, and Additive Technologies (CiMAT) at the University of Warwick under the EPRSC Project Number (EP/T02593X/1).


\subsection{Metabolite extraction and \ac{lcms} settings}
Aliquots of 10mg from each homogenised sample were extracted on ice in 400\(\mu\)L 60\%\ac{lcms} grade methanol, vortexed for 30 seconds every 10 minutes for 30 minutes, returning to ice in between, sonicated for 15 minutes in ice-water, and centrifuged at 13000rpm for 10 minutes at 4 $^{\circ}$C. The supernatant was transferred to a clean 2ml Eppendorf. The extraction process was then repeated. After overnight storage at 4 $^{\circ}$C, samples were filtered through a PVDF syringe filter, and the filtrate was transferred into a glass \ac{lcms} vial for analysis. Quality control (QC) samples were generated by pipetting 10\(\mu\)L from each sample into an \ac{lcms} vial. Analysis was conducted using the Dionex UltiMate 3000 UHPLC system and Agilent Eclipse Plus C18 UPLC column with outflow routed to a Bruker MaXis II Q-TOF with an electrospray source. Samples were run in both positive and negative ion mode. Samples and blanks were randomised.

\subsection{Data processing, availability, and statistical Analysis}
Raw data were converted to the mzXML format using Bruker Compass DataAnalysis (v4.4 SR1). Samples C12-2 and X12-4 were removed from further analysis, as possible contaminants were identified in raw Base Peak Chromatograms. To generate XCMSOnline  (v2.7.2) \parencite{Gowda2014} starting criteria, the mzXML files were processed in R (v4.3.1) \parencite{R} using IPO (v1.10.0) \parencite{Libiseller2015}. After, samples were grouped by treatment and time, with each sample group containing four biological replicates and peaks were identified using  XCMSOnline (v2.7.2) \parencite{Gowda2014}. (See \href{https://github.com/JamiePike/UntargetedMetabolomics/tree/main/NovDec22/XCMS}{GitHub Repo} for XCMS settings). Features with a retention time of less than 60 seconds, as well as predicted adducts (CAMERA v1.34.0), were removed from the data set before statistical analysis. The blank samples were also removed from the statistical analysis.

For statistical analysis, samples were grouped by time point and/or treatment and analyses were performed using MetaboAnalyst (v6.0) (\href{https://www.metaboanalyst.ca/}{https://www.metaboanalyst.ca/}), and custom R (v4.3.1) \parencite{R} scripts (See: \href{https://github.com/JamiePike/UntargetedMetabolomics/tree/main}{GitHub Repository}). Peak areas were normalised to the Na(NaCOOH)3 adduct ($m/z=226.9521$) of the sodium formate internal standard. Preliminary analysis identified some outliers in the dataset (n=7, see Appendix C\ref{apx:outliers}). The majority of outlier samples were from the third time point (n=5). Therefore, only the first and second time points were analysed. All scripts, logs, and a detailed breakdown of the approach used in this analysis are available via the associated GitHub repository \href{https://github.com/JamiePike/UntargetedMetabolomics}{ (https://github.com/JamiePike/UntargetedMetabolomics)}. 

\newpage
\section{Results}

\subsection{Symptom development from \acl{Focub4} infection progressed slower than \acl{xvm} infection.}

Internal and external symptoms were evaluated from \ac{Focub4} inoculated plants at 15, 18, and 21 \ac{dpi}, and \ac{xvm} inoculated, drought stress, and mock plants at 7, 10, and 13 \ac{dpi}. Symptom scores demonstrated variability across assessment types (external and internal). External symptom scores were consistently higher in plants experiencing drought stress compared to those inoculated with \ac{Focub4} and \ac{xvm} (Figure: \ref{fig:SymptomDev}a). The average external symptom score for drought-stressed plants started at 2.75 at the initial time point and progressed to 4 by the third time point. Limited variation was observed between the drought-stressed plants across the time points (Figures \ref{fig:DroFirstTimeBLQs},  \ref{fig:DroSecondTimeBLQs}, and \ref{fig:DroThirdTimeBLQs}). Internal symptom scores were highest in \ac{xvm}-inoculated plants, showing an increase from 2 at the first time point to 4.75 at the third time point (Figure: \ref{fig:SymptomDev}b). Interestingly, \ac{Focub4}-inoculated and drought-stressed plants displayed similar internal symptom scores.

\begin{figure}[h!]
    \centering
    \includegraphics[width=\textwidth]{Figures/Combined_Sympotoms_plot.png}
    \caption[Symptom development scores of 'Grand Naine' plants infected with \acl{Focub4} or \acl{xvm}, or subjected to drought stress.]{\textbf{Symptom development scores of 'Grand Naine' plants infected with \acf{Focub4} or \acf{xvm}, or subjected to drought stress.} \textbf{a}) External symptom development. \textbf{b}) Internal symptom development. Error bars show standard deviation.}
    \label{fig:SymptomDev}
\end{figure}

Plants inoculated with \ac{xvm} displayed rapid development of external symptoms compared to \ac{Focub}, with an initial average symptom score of 1, increasing to 3.2 by the third time point. \ac{Focub4}-inoculated plants exhibited an initial average external symptom score of 1.75, which increased to 2.5 by the third time point. At the second time point, \ac{xvm}-inoculated and \ac{Focub4}-inoculated plants displayed the same average external symptom score (Figure: \ref{fig:SymptomDev}a), but some differences were present in the types of symptoms observed. Foliage from \ac{xvm}-inoculated plants generally appeared wilted, whereas foliage \ac{Focub4}-inoculated plants was more chlorotic (Figures \ref{fig:XvmSecondTimeBLQs} and \ref{fig:FocSecondTimeBLQs}).
 
There was a large variation in individual external symptom scores by the third time point, particularly in \ac{xvm}-inoculated plants (Figure \ref{fig:XvmThirdTimeBLQs}), with sample X15-4, displaying less severe symptoms compared to other plants from the \ac{xvm} treatment group. Interestingly, a sucker had developed on \ac{xvm}-inoculated plant, x15-3, which displayed no symptoms (suckers were not included in symptom scoring). In comparison to \ac{xvm}-inoculated plants, symptoms were not as well developed in the \ac{Focub4}-inoculated plants (Figure \ref{fig:FocThirdTimeBLQs}). All drought plants exhibited severe symptoms by the third time point (Figure \ref{fig:DroThirdTimeBLQs}).

\begin{figure}[ph!]
    \centering
    \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/FirstTimePointXanthomonasBLQs.pdf}
        \caption{}
        \label{fig:XvmFirstTimeBLQs}
    \end{subfigure}
     \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/FirstTimePointFusariumBLQs.pdf}
        \caption{}
        \label{fig:FocFirstTimeBLQs}
    \end{subfigure}
         \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/FirstTimePointDroughtBLQs.pdf}
        \caption{}
        \label{fig:DroFirstTimeBLQs}
    \end{subfigure}
         \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/FirstTimePointControlBLQs.pdf}
        \caption{}
        \label{fig:ConFirstTimeBLQs}
    \end{subfigure}
    \caption[RGB images of 'Grande Naine' banana plants at the first sample time points.]{\textbf{RGB images of 'Grande Naine' banana plants at the first sample time points.}
    \textbf{\subref{fig:XvmFirstTimeBLQs}}) \acl{xvm}-inoculated plants at 7 \acl{dpi}.
    \textbf{\subref{fig:FocFirstTimeBLQs}}) \acl{Focub4}-inoculated plants at 15 \acl{dpi}.
    \textbf{\subref{fig:DroFirstTimeBLQs}}) Drought-stressed plants at 7, no-watering.
    \textbf{\subref{fig:ConFirstTimeBLQs}}) Mock-inoculated plants at 7 \ac{dpi}.
    Plant labels are provided in white text and correspond to sample labels used in \acl{um} analysis.
    }
    \label{fig:FirstTimePointSymptoms}
\end{figure}

\begin{figure}[ph!]
    \centering
    \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/SecondTimePointXanthomonasBLQs.pdf}
        \caption{}
        \label{fig:XvmSecondTimeBLQs}
    \end{subfigure}
     \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/SecondTimePointFusariumBLQs.pdf}
        \caption{}
        \label{fig:FocSecondTimeBLQs}
    \end{subfigure}
         \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/SecondTimePointDroughtBLQs.pdf}
        \caption{}
        \label{fig:DroSecondTimeBLQs}
    \end{subfigure}
         \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/SecondTimePointControlBLQs.pdf}
        \caption{}
        \label{fig:ConSecondTimeBLQs}
    \end{subfigure}
    \caption[RGB images of 'Grande Naine' banana plants at the second sample time points.]{\textbf{RGB images of 'Grande Naine' banana plants at the second sample time points.}
    \textbf{\subref{fig:XvmSecondTimeBLQs}}) \acl{xvm}-inoculated plants at 10 \acl{dpi}.
    \textbf{\subref{fig:FocSecondTimeBLQs}}) \acl{Focub4}-inoculated plants at 18 \acl{dpi}.
    \textbf{\subref{fig:DroSecondTimeBLQs}}) Drought-stressed plants at 10, no-watering.
    \textbf{\subref{fig:ConSecondTimeBLQs}}) Mock-inoculated plants at 10 \ac{dpi}.
    Plant labels are provided in white text and correspond to sample labels used in \acl{um} analysis.
    }
    \label{fig:SecondTimePointSymptoms}
\end{figure}

\begin{figure}[ph!]
    \centering
    \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/ThirdTimePointXanthomonasBLQs.pdf}
        \caption{}
        \label{fig:XvmThirdTimeBLQs}
    \end{subfigure}
     \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/ThirdTimePointFusariumBLQs.pdf}
        \caption{}
        \label{fig:FocThirdTimeBLQs}
    \end{subfigure}
         \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/ThirdTimePointDroughtBLQs.pdf}
        \caption{}
        \label{fig:DroThirdTimeBLQs}
    \end{subfigure}
         \begin{subfigure}[b]{\textwidth}
        \includegraphics[width=\textwidth]{Figures/ThirdTimePointControlBLQs.pdf}
        \caption{}
        \label{fig:ConThirdTimeBLQs}
    \end{subfigure}
    \caption[RGB images of 'Grande Naine' banana plants at the third sample time points.]{\textbf{RGB images of 'Grande Naine' banana plants at the third sample time points.}
    \textbf{\subref{fig:XvmThirdTimeBLQs}}) \acl{xvm}-inoculated plants at 13 \acl{dpi}.
    \textbf{\subref{fig:FocThirdTimeBLQs}}) \acl{Focub4}-inoculated plants at 21 \acl{dpi}.
    \textbf{\subref{fig:DroThirdTimeBLQs}}) Drought-stressed plants at 13, no-watering.
    \textbf{\subref{fig:ConThirdTimeBLQs}}) Mock-inoculated plants at 13 \ac{dpi}.
    Plant labels are provided in white text and correspond to sample labels used in \acl{um} analysis.
    }
    \label{fig:ThridTimePointSymptoms}
\end{figure}

\subsection{Feature distribution between treatment groups and timepoints}

A total of 3,931 aligned features were detected across all samples, divided into groups by treatment and time point, using XCMSOnline (v2.7.2) \parencite[]{Gowda2014} (following the retention time and adduct filtering). Of the 3,931 aligned features, 633 were identified as statistically significant ($p \le0.05$) by a one-way \ac{anova}. Hierarchical clustering of samples and the 633 significant features ($p \le0.05$) revealed that sample groups clustered by time (Figure \ref{fig:Sig657FeaturesRedSamples}), apart from the \ac{xvm}-inoculated samples from the first time point (7 \ac{dpi}). Samples from the same treatment group did not all cluster together when using the 633 significant features ($p \le0.05$).

\begin{figure}[h!]
    \centering
    \includegraphics[width=0.7\textwidth]{Figures/Sig633FeaturesRedSamplesRedGroups_ForThesis.pdf}
    \caption[Heatmap showing hierarchical clustering of samples based on 633 normalised significant feature peak intensities ($p \le0.05$).]{\textbf{Heatmap showing hierarchical clustering of samples based on 633 normalised significant feature peak intensities ($p \le0.05$).} The distance measure was Euclidean and the ward.D algorithm was used for clustering. Class shows the treatment and time groups. Value shows the normalised peak intensity. Features were normalised to the sodium formate Na(NaCOOH)3 adduct ($m/z=226.9521$), log-transformed and scaled (Pareto scaling). Figure generated using MetaboAnalyst (v6.0).}
    \label{fig:Sig657FeaturesRedSamples}
\end{figure}

As we aim to identify features specific to the treatment group, not time, samples were separated into individual time points for the following analyses. Of the 3,931 peaks detected across all samples and time points, 232 were identified as significant ($p \le0.05$) using a one-way \ac{anova} in the first time point. Samples from the first time point mostly clustered into their respective treatment groups when hierarchical clustering was performed using these 232 significant features ($p \le0.05$) (Figure \ref{fig:Sig232FeaturesRedSamples}). One sample from the drought treatment, D9.4, clustered with the Fusarium samples from the first time point. Additionally, sample C9.2, from the control treatment, did not cluster with the control group, with much lower peak intensities recorded for many of the significant features from this sample when compared to the other control samples.  

\begin{figure}[h!]
    \centering
    \includegraphics[width=0.7\textwidth]{Figures/Sig232Features_heatmap_9_dpi72-ForThesis.pdf}
    \caption[Heatmap showing hierarchical clustering of samples based on 232 normalised significant feature peak intensities ($p \le0.05$) from the first time point.]{\textbf{Heatmap showing hierarchical clustering of samples based on 232 normalised significant feature peak intensities ($p \le0.05$) from the first time point.} The distance measure was Euclidean and the ward.D algorithm was used for clustering. Class shows the treatment and time groups. Value shows the normalised peak intensity. Features were normalised to the sodium formate Na(NaCOOH)3 adduct ($m/z=226.9521$), log-transformed and scaled (Pareto scaling). Figure generated using MetaboAnalyst (v6.0).}
    \label{fig:Sig232FeaturesRedSamples}
\end{figure}

In the second time point, only 171 significant features ($p \le0.05$) were identified (unequal group variance) . When hierarchical clustering was performed on these significant 171 features, samples clustered based on the treatment group (Figure \ref{fig:Sig171FeaturesRedSamples}). However, one sample from the drought treatment, D12.2, clustered with the samples from the \ac{xvm} treatment group. \Ac{plsda} analysis of the 171 significant features from the second time point, revealed clear discrimination of the \ac{Focub4}, \ac{xvm}, drought-stressed, and mock-inoculated plants based on their metabolite profiles (Figure \ref{fig:FeaturesTimePoint_plsda}). Of the 232 and 171 significant features identified in the samples from the first time point and second time point, respectively, only six features were shared between time points (Figure \ref{fig:FeaturesTimePoint_Venn}). As a total of six shared features is low, we checked for potentially shared features by identifying features with similar retention times ($\pm1$ second), and $m/z$ values that differ by the nominal masses of common \ac{lcms} adducts, (e.g. M+H, M+Na, M+K). This identified a further set of 19 potentially shared features. 


\begin{figure}[!hptb]
    \centering
    \begin{subfigure}[b]{0.42\textwidth}
        \includegraphics[width=\textwidth]{Figures/heatmapofsigfeatures-ForThesis-Long.pdf}
      \caption{}
      \label{fig:Sig171FeaturesRedSamples}
    \end{subfigure}
    \hfill
    \begin{minipage}[b]{0.57\textwidth}
      \begin{subfigure}[b]{\linewidth}
      \begin{subfigure}[b]{\linewidth}
	    \includegraphics[width=\textwidth]{Figures/PLSDA_SigFeaturesRedSamplesRedGroupsSecondTimePoint.pdf}
        \caption{}
        \label{fig:FeaturesTimePoint_plsda}
      \end{subfigure}
	    \includegraphics[width=\textwidth]{Figures/SharedFeaturesVenn_Time.png}
        \caption{}
        \label{fig:FeaturesTimePoint_Venn}
      \end{subfigure}\\[\baselineskip]
    \end{minipage}
    \caption[Distribution of features at the second time point.]{\textbf{Distribution of significant features at the second time point.}
    \textbf{\subref{fig:Sig171FeaturesRedSamples}}) Heatmap showing hierarchical clustering of samples based on 171 normalised significant feature peak intensities ($p \le0.05$). The distance measure was Euclidean and the ward.D algorithm was used for clustering. Class shows the treatment and time groups. Value shows the normalised peak intensity. Features were normalised to the sodium formate Na(NaCOOH)3 adduct ($m/z=226.9521$), log-transformed and scaled (Pareto scaling). 
    \textbf{\subref{fig:FeaturesTimePoint_plsda}}) \acl{plsda} score plot of significant features at the second time point. 
    \textbf{\subref{fig:FeaturesTimePoint_Venn}}) Shared significant features ($p \le0.05$) between the first two time points. T1 = first-time point, T2 = second-time point.
    Figures \subref{fig:Sig171FeaturesRedSamples} and \subref{fig:FeaturesTimePoint_plsda} were generated using MetaboAnalyst (v6.0). 
    }
    \label{fig:SecondTimePointSigFig}
\end{figure}


\subsection{Unique features identified in the \acl{Focub4} at the second time point.}

As the symptoms scores for \ac{Focub4} and \ac{xvm} are the same at the second time point (1.5), we decided to focus on features that can help to discriminate between the two wilts at this time point. Though two mock-inoculated samples is too low a sample number to conclude a meaningful difference, we conducted pairwise comparisons of the mock-inoculated samples against the remaining sample groups to identify features of interest. Using the 171 significant features ($p \le0.05$) identified from the second time point, Students' t-tests (unequal group variance) identified 63 features that are of interest ($p \le0.05$) in \ac{Focub4}-inoculated plants compared to mock-inoculated plants (Figure  \ref{fig:FocVsCon_t-test}). In both the pairwise comparisons of \ac{xvm}-inoculated to mock-inoculated plants and drought-stressed to mock-inoculated plants, 48 features of interest were identified (Figure  \ref{fig:XvmVsCon_t-test} and \ref{fig:DroVsCon_t-test}).
Of all the features of interest (n=159), 25 are shared between all treatments (Figure  \ref{fig:PairwiseVenn-SecondTimePoint}). The \ac{Focub4}-inoculated plants compared to mock-inoculated plants had 18 unique features of interest, whereas only eight and six unique features of interest were identified in the \ac{xvm}-inoculated to mock-inoculated plants and drought-stressed to mock-inoculated plants comparisons, the remaining 26 features of interest were shared between two of the pairwise comparisons (Figure  \ref{fig:PairwiseVenn-SecondTimePoint}).

\textcolor{red}{ARE THESE UNIQUE/SHARED FEATURES ADDUCTS OF SAME METABOLITES?}

\begin{figure}[ph!]
    \centering
    \begin{subfigure}[b]{0.49\textwidth}
        \includegraphics[width=\textwidth]{Figures/Sig171FeaturesRedGroupsDroVsConSecondTimePoint_t-test.png}
        \caption{}
        \label{fig:DroVsCon_t-test}
    \end{subfigure}
    \begin{subfigure}[b]{0.49\textwidth}
        \includegraphics[width=\textwidth]{Figures/Sig171FeaturesRedSamplesXvmVsConSecondTimePoint_t_test.png}
        \caption{}
        \label{fig:XvmVsCon_t-test}
    \end{subfigure}
    \begin{subfigure}[b]{0.49\textwidth}
        \includegraphics[width=\textwidth]{Figures/Sig171FeaturesRedSamplesFocVsConSecondTimePoint_t_test.png}
        \caption{}
        \label{fig:FocVsCon_t-test}
    \end{subfigure}
    \begin{subfigure}[b]{0.49\textwidth}
        \includegraphics[width=\textwidth]{Figures/Pairwise_SharedFeaturesVenn_SecondTimePoint.png}
        \caption{}
        \label{fig:PairwiseVenn-SecondTimePoint}
    \end{subfigure}
    \caption[Distribution of significant features at the second time point.]{\textbf{Distribution of significant features ($p \le0.05$) at the second time point.}
    Utilising the 171 significant features ($p \le0.05$) identified through the one-way \ac{anova} conducted on features at the second time point, each treatment was compared to the mock-inoculated control for pairwise analysis.
    \textbf{\subref{fig:DroVsCon_t-test}}) shows features of interest ($p \le0.05$) from drought-stress vs control pairwise analysis;
    \textbf{\subref{fig:XvmVsCon_t-test}}) shows features of interest ($p \le0.05$) from the \acl{xvm} vs control pairwise analysis; 
    \textbf{\subref{fig:FocVsCon_t-test}}) shows features of interest ($p \le0.05$) from the \acl{Focub4} vs control pairwise analysis. The red circles represent features above the $p \le0.05$ threshold. The $p$ values are transformed by $-log10$ so that the features of interest are plotted higher on the graph.
    \textbf{\subref{fig:PairwiseVenn-SecondTimePoint}}) Venn diagram depicting shared/unique features of interest ($p \le0.05$) in drought-stressed (blue), \ac{xvm}-inoculated (yellow), or \ac{Focub4}-inoculated (red) plants compared to mock-inoculated plants at the second time point. 
    Panels \subref{fig:DroVsCon_t-test}, \subref{fig:XvmVsCon_t-test}, and \subref{fig:FocVsCon_t-test} were generated using MetaboAnalyst (v6.0).  
    }
    \label{fig:enter-label}
\end{figure}

\subsection{Features of interest and putative identification.}

\subsubsection{Positive mode features of to distinguish abiotic and biotic stress}

\textbf{P1}

\textbf{P2}

\textbf{P3}...

\subsubsection{Positive mode features of interest  from \ac{Focub} and putative identification.}

%\subsection{Multispectral imaging as an approach for early symptom detection.}

%Multispectral imaging represents a promising approach for the early detection of symptoms in plant pathology. Despite our rigorous analysis, we were unable to discern any significant differences between the various treatments before the naked eye.  \textcolor{red}{I need to show the images here and talk about the lack of obvious difference.} This suggests that further refinement or complementary techniques may be necessary to enhance the sensitivity of multispectral imaging for early symptom detection in our experimental setup. 

%\subsubsection{Multispectral imaging does not distinguish symptoms between different wilting stresses.}

%One of the primary objectives of this study was to investigate whether multispectral imaging could differentiate between different types of wilting stresses. However, our findings indicate that multispectral imaging did not effectively distinguish between the symptoms induced by various wilting stresses. \textcolor{red}{Here I can talk about the time of symptom development, but need to remember for discussion that growers are unlikely to image over time and the environmental factors will affect the speed of disease progression too quickly to make this a viable option.}
%This underscores the complexity of symptom manifestation and suggests the need for alternative approaches or additional parameters to improve the discriminatory power of multispectral imaging in distinguishing between different types of wilting stresses.

\newpage
\section{Discussion and conclusion}

Low sample number - the strength of statistics? Are data within the confidence interval? 

A lot of variation between times, but not as much between treatments. 

Much use in finding a feature if it is only present for a day or so? Chosen biomarker features need to be found as a discriminatory metabolite at all time points...


"Our study provides a wealth of potential information for future investigation and are the only set of replicated, unbiased LC-MS metabolite data from verified tolerant and susceptible European ash and these are available with associated metadata in Metabolights24,25 [MTBLS372]." SAMBLES PAPER: https://www.nature.com/articles/sdata2017190