
\section{Introduction}

Alongside the genomic analysis, we developed and employed other -omics techniques to improve our understanding of some important biological parameters of \ac{Focub}. The application of  a multi-omics approach to investigate plant diseases has become increasingly prevalent in recent years \parencite{Crandall2020}. We employed phenomic analysis including multispectral and RGB imaging and untargeted metabolomics. 

Multispectral and RGB imaging studies in banana have primarily been establish for disease detection and diagnosis. \textcite{Johansen2014} developed an automatic banana identification software to aid in Banana Bunch Top Virus inspection and \textcite{Liao2018} employed hyperspectral images and machine learning to diagnose Banana Streak Virus at earlier stages of infection. \textcite{Ochoa2016} designed a hyperspectral imaging system for disease scanning on banana plants focusing on Black Sigatoka. Unmanned aerial vehicles (UAVs), RGB imaging, and Artificial Neural Networks have also employed in the monitoring of Yellow Sigatoka (Calou, et al., 2020). Ye, et al. (2020a) demonstrated that UAV-based multispectral imagery can be used to diagnose Fusarium wilt, observing statistically significant differences (p > 0.05) between healthy and diseased plants using six different vegetation indices (VIs). RGB images alongside ML and artificial intelligence (AI) tools have been used to detect and differentiate between different disease in banana, including Foc, with the AI technology now available as a mobile phone application for banana growers (Selvaraj et al., 2019, Selvaraj et al., 2020).
It is clear that multispectral, hyperspectral, and RGB images can be used to diagnose Fusarium wilt, along with many other abiotic and biotic stresses, in banana. What is not clear, however, is what happening from a biological perspective to cause the differences in spectra observed, and whether spectra are different when comparing biotic and abiotic stresses. As Foc is a wilting pathogen it is important to ensure that Foc infection can be differentiated from drought stress and other wilting pathogens, such as Xanthomonas campestris pv. musacearum (Xvm). 
Something about pathogen progression internally and symptom development â€“ then I can write about CT scanning here too!




Metabolomics, another vital "omics" field used in biological sciences, is the comprehensive analysis of the spectrum of metabolites within a biological system \parencite{Klassen2017}. Frequently used to track disease progression and pinpoint potential novel metabolites associated with disease and disease susceptibility, metabolomics has become a useful tool in disease diagnostics. As metabolites mirror downstream expression of the genome, transcriptome, and proteome, they offer an intimate snapshot of an organism's phenotype at a specific stage of infection and scrutinising metabolic variations in infected, resistant, and healthy individuals can unearth disease, susceptibility, or resistance biomarkers. Furthermore, biomarkers may be used for the early identification of diseases, potentially underpinning a pivotal disease management strategy. 

Metabolomics studies are typically classified into two main categories: targeted and untargeted analyses. Targeted metabolomics (TM) approaches identify a specifically selected set of compounds for analysis. This approach is usually employed when precise quantitative analysis is necessary, often requiring complex extraction protocols. When employing an \ac{um} approach, researchers aim to identify a wide range of features without predefined targets. This facilitates the discovery of new and diverse compounds, including previously unknown compounds and metabolites. \ac{um} typically employs simpler extraction and detection procedures compared to targeted studies. However, it generates highly intricate data, which comes with an increased challenge of false discoveries. Consequently, \ac{um} demands substantial effort in data analysis and interpretation.

Now emerging as a useful tool in plant pathology, \ac{um} has been employed in the study of infection as well as plant resistance and susceptibility \parencite{Allwood2021}. Notably, \textcite{Garcia2018} harnessed \ac{lcms} to pinpoint biomarkers indicative of \textit{Phythopthora infestans} infection in tomato plants, even in asymptomatic cases. Similarly, \textcite{Sambles2017} and \textcite{Sidda2020} used \ac{um} to identify certain secoiridoid glycosides as discriminatory metabolites signifying susceptibility to ash die-back in both UK and Danish ash trees (\textit{Fraxinus excelsior}).

\textcite{Kasote2020} employed a compartmentalised TM and\ac{um} strategy to differentiate watermelon (\textit{Citrullus vulgaris}) plants exhibiting symptoms of \textit{Fusarium} wilt, caused by \ac{Fon}), from their asymptomatic counterparts, while also distinguishing them from healthy plants of distinct varieties. Using this approach, the authors were able to identify biomarkers associated with the progression of Fusarium wilt across various watermelon varieties. They showed that the metabolic profiles of \textit{Fusarium} wilt-infected plants exhibit distinct variations depending on their genotype, as well as differences between leaf and stem tissues compared to the root. Phytohormones such as jasmonic acid-isoleucine (JA-Ile) and methyl jasmonate (MeJA) accumulated in resistant varieties, whereas indole-3-acetic acid (IAA) was identified in all resistant lines 16 days after inoculation. The authors suggest that IAA can be used as a potential biomarker of \ac{Fon} infection in watermelon. However, as IAA, as well as the other phytohormones identified,  are commonly associated with immune responses, one has to question how effective they will be as a \ac{Fon} specific biomarker. The varying levels of amino acids (Arg, Asp, Cit, His, Val, and Lys) and the phenolic acid, Phthalic acid (PHA) also offer valuable insights into the interaction between \ac{Fon} and watermelon plants. However, it is imperative to conduct further comprehensive investigations to determine the precise roles of these distinctive metabolites in the development of \textit{Fusarium} wilt and their contribution and specificty to watermelon plant resistance against \ac{Fon}.

Most research into the metabolic composition of bananas primarily focuses on the fruit and its relationship with diet. Only a limited number of studies have delved into the metabolic profiles of bananas affected by \ac{Focub}. In a study conducted by \textcite{Li2013c}, the virulence of various \ac{Focub} isolates was assessed, and LC/MS/MS analysis was used to quantify the presence of two mycotoxins commonly associated with \ac{Focub}, namely beauvericin and fusaric acid, in different parts of banana plants such as roots, fruits, pseudostems, and foliage. Their findings revealed a strong correlation between virulence and the accumulation of these toxins. Additionally, they investigated the occurrence of these toxins in field-grown plants displaying symptoms of \ac{Focub} infection and found that, while the toxins were present in the fruit, their levels were too low to pose any significant risk to human or animal health.

To the best of our knowledge, there have been no studies to date that have employed \ac{um} to comprehensively analyse the metabolic profile of \ac{Focub}-infected banana plants. One objective of this thesis is to compare the data generated from metabolomics studies with imaging data, to shed light on the biological processes contributing to the observed differences during imaging. This approach also aimed to identify specific markers associated with each stress condition. 

\newpage
\section{Materials and Methods}

\subsection{Sample collection}
Samples from four treatments were collected. Plants were inoculated with \ac{Focub} or \ac{xvm} or were exposed to drought stress (no watering from the time of \ac{xvm} inoculation). A water mock inoculation treatment group was also established (see ~\ref{sec:Chapter4_MM}). 
Sample collection was staggered to ensure that symptom scores were similar between treatments. Samples were collected plants inoculated with \ac{Focub4} at 15, 18 and 21 \ac{dpi}, or from plants inoculated with \ac{xvm}, as well as the drought stress and water inoculated plants at 7, 10, and 13 \ac{dpi}. Three leaves were collected per plant, with three plants sampled from each treatment group at each these time points. A 75mm by 25mm section of lamina on either side of the midrib was excised from the base of each leaf, snap-frozen in liquid N2, lyophilised, and ground to a powder. The powdered lamina were then homogenised to generate a single leaf sample for each plant. 

\subsection{Metabolite Extraction and LC-MS Setting}
Aliquots of 10mg from each homogenised sample were extracted on ice in 400\(\mu\)L 60\% LC-MS grade methanol, vortexed for 30 seconds every 10 minutes for 30 minutes, returning to ice in between, sonicated for 15 minutes in ice-water, and centrifuged at 13000rpm for 10 minutes at 4 $^{\circ}$C. The supernatant was transferred to a clean 2ml Eppendorf. The extraction process was then repeated. After overnight storage at 4 $^{\circ}$C, samples were filtered through a PVDF syringe filter, and the filtrate was transferred into a glass LC-MS vial for analysis. Quality control (QC) samples were generated by pipetting 10\(\mu\)L from each sample into an LC-MS vial. Analysis was conducted using the Dionex UltiMate 3000 UHPLC system and Agilent Eclipse Plus C18 UPLC column with outflow routed to a Bruker MaXis II Q-TOF with an electrospray source. Samples were run in both positive and negative ion mode. Samples and blanks were randomised.

\subsection{Data Processing and Statistical Analysis}
Raw data were converted to the mzXML format using Bruker Compass DataAnalysis 4.4 SR1. The mzXML files were processed in R using the packages IPO (v1.10.0) \parencite{Libiseller2015}, XCMS (v3.6.2) \parencite{Benton2010} and CAMERA (v1.40.0) \parencite{Kuhl2012}   (See \href{https://github.com/JamiePike/UntargetedMetabolomics/tree/main/NovDec22/XCMS}{GitHub Repo} for XCMS settings). Samples were initially only grouped by treatment and time, with each sample group containing four biological replicates. 

Samples were grouped by time point and/or treatment and analyses were performed using MetaboAnalyst (v6.0) (\href{https://www.metaboanalyst.ca/}{https://www.metaboanalyst.ca/}), and custom R scripts (See: \href{https://github.com/JamiePike/UntargetedMetabolomics/tree/main}{GitHub Repo}). Peak areas were normalised to the Na(NaCOOH)3 adduct (m/z=226.9515) of the sodium formate internal standard. All scripts, logs, and a detailed breakdown of approach used in these analysis are available via the associated GitHub repository \href{https://github.com/JamiePike/UntargetedMetabolomics}{ (https://github.com/JamiePike/UntargetedMetabolomics)}. 

\newpage
\section{Results}
\subsection{Untargeted metabolomics: the banana-pathogen metabolome}




\newpage
\section{Discussion and conclusion}
