% Please add the following required packages to your document preamble:
% \usepackage{multirow}
% \usepackage{graphicx}
% \usepackage{lscape}


\begin{landscape}
\begin{table}[]
\captionsetup{width=\linewidth} % Set the width of the caption to match the table
\caption{[Overall alignment rate of all raw reads from each TNAU isolates to fungal and bacterial reference species]{\textbf{Overall alignment rate of all raw reads from each TNAU isolates to fungal and bacterial reference species.}} Overall alignment rate determined by Bowtie2 (version 2.4.5). Reference assemblies were downloaded from GenBank.}
\label{tab:RawReadMapping}
\resizebox{\columnwidth}{!}{%
\begin{tabular}{lccccccc}
\multicolumn{1}{c}{\multirow{2}{*}{Species}} & \multicolumn{3}{c}{Bowtie2 Alignment Rate} & \multirow{2}{*}{Strain} & \multirow{2}{*}{GenBank  Accession} & \multirow{2}{*}{No. of Contigs} & \multirow{2}{*}{Contig N50 (Mb)} \\ \cline{2-4}
\multicolumn{1}{c}{}                         & S6           & S16          & S32          &                         &                                     &                                 &                                  \\ \hline
\textit{F. oxysporum f. sp. cubense (TR4)}   & 8.72\%       & 53.81\%      & 15.69\%      & UK0001                  & GCA\_007994515.1                    & 15                              & 4.49                             \\
\textit{F. sacchari}                         & 5.24\%       & 68.65\%      & 22.49\%      & FS66                    & GCA\_017165645.1                    & 48                              & 1.97                             \\
\textit{Stenotrophomonas maltophilia}        & 49.32\%      & 0.01\%       & 53.93\%      & NCTC10258               & GCA\_900475405.1                    & 1                               & 4.5                              \\ \hline
\end{tabular}%
}
\end{table}
\end{landscape}
