%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%CODE OUTPUT STYLE
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\lstset{basicstyle=\ttfamily,
  showstringspaces=false,
  commentstyle=\color{red},
  keywordstyle=\color{blue}
}

\lstdefinestyle{customc}{
  belowcaptionskip=1\baselineskip,
  breaklines=true,
  showstringspaces=false,
  basicstyle=\footnotesize\ttfamily,
  keywordstyle=\bfseries\color{green!40!black},
  commentstyle=\itshape\color{purple!40!black},
  identifierstyle=\color{blue},
  stringstyle=\color{orange},
}

%Stops the script running off the page
\lstset{escapechar=@,style=customc}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%CODE SECTIONS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{GCTrimmer.py}\label{apx:gcTrimmer.py}
Script used to separate contigs in the TNAU assemblies into separate FASTA files based on GC\% threshold. 

\begin{lstlisting}[language=Python, caption=Script to separate contigs based on GC\% threshold]
#Extracts sequences from a FASTA file and order in to two different FASTAs depending on GC content

###############################################
#Set up and import modules.
import re, sys, colorama
from datetime import date
from tqdm import tqdm
from colorama import Fore
from Bio import SeqIO, SeqUtils
from Bio.SeqUtils import GC
from Bio.SeqRecord import SeqRecord
###############################################
#Establish input file (FASTA) and gc Threshold.
infile = sys.argv[1]
gcContentThreshold = sys.argv[2]
###############################################
#Establish colour reset.
colorama.init(autoreset=True)
###############################################
# Establish Count for sequences above and below threshold.
SeqAboveCount = 0
SeqBelowCount = 0

###############################################
#Progress:
#Record date
today = date.today()
startTime = today.strftime("%B %d, %Y")
#Create progress bar
num = len([1 for line in open(infile) if line.startswith(">")])
print("="*80)
print(f'Running gcTrimmer.py.\n{startTime}.\n\nInput File:\t\t{infile}\nNumber of Contigs:\t{num}\nGC threshold:\t\t{gcContentThreshold}%\n')

#Start progress bar
with tqdm(total=num) as pbar:
###############################################
    #Open two .fasta files using the name of the infile.
    with open(f'{infile}_GCcontentBelow{gcContentThreshold}perc.fasta'.format(), "a") as LessThanFile, open(f'{infile}_GCcontentAbove{gcContentThreshold}perc.fasta'.format(), "a") as GreaterThanFile, open(f'gcContentReport.txt'.format(), "a") as report  :

    #Parse and loop through the FASTA input file. Search each scaffold/contig for sequences with length => than the desired GC content and deposit them in the FASTA
    #containing the less than the desired gc content. If the sequence gc content is greater than the gc content threshold, then it is added to the "greater than" FASTA file.
        for sequence in SeqIO.parse(infile, "fasta"):
            print("="*10, file=report, sep="\n")
            gccontent = (sequence.seq.count('G') + sequence.seq.count('C')) / len(sequence) * 100
            pbar.update(1)
            if gccontent < int(gcContentThreshold):
                print(f'{sequence.description} is below the threshold GC content. GC content is {Fore.RED}{gccontent}{Fore.BLACK}.\n', file=report, sep="\n")
                print(sequence.format("fasta"), file=LessThanFile, sep="\n")
                SeqBelowCount = SeqBelowCount + 1
            else:
                print(f'{sequence.description} is above the threshold GC content. GC content is {Fore.GREEN}{gccontent}{Fore.BLACK}.\n', file=report, sep="\n")
                print(sequence.format("fasta"), file=GreaterThanFile, sep="\n")
                SeqAboveCount = SeqAboveCount + 1


#Print the total number of sequences above and below the GC threshold.

print(f'\nThe number of sequences above the GC threshold: {Fore.GREEN}{SeqAboveCount}{Fore.BLACK}.\nThe number of sequences below the GC threshold: {Fore.RED}{SeqBelowCount}{Fore.BLACK}.')
print("="*80)    
\end{lstlisting}

\section{ContaminatFilter.sh}\label{apx:ContamFilter}
\begin{lstlisting}[language=Python, caption=bash script to filter contigs from assembly that are predicted to belong to a non-\textit{Fusarium} genus by Blobtools.]
#!/bin/bash

#Jamie Pike
#Command for filtering conatminant blobtools hits. This groups all of the standard BlobTools commands I used to remove a contaminat contigs analysis. 

python -c "print('=' * 75)"
echo "Blobtools Contaminant Filter"
echo "----------------------------"
echo $(date)
echo "Usage: ContaminantFilter.sh <FASTA file> <blobtools json file> <outfile prefix>"
python -c "print('=' * 75)"

infile=${1?Please provide the assembly input fasta.} #Input Assembly.
json=${2?Please provide a BlobTools json file.} #Input BAM.
prefix=${3?Please provide a prefix for you outputs.} 

########################
#Escape the script if there are any errors. 
set -e 

echo "Creating species table..."
#Use BlobTools view to generate a blobtools table.txt file for filtering.
blobtools view -i ${json} -r species -o ${prefix}

echo "Filetering for Fusarium and no-hit only contigs..."
#Use grep to extact only the Fusarium and no-hit lines from the hit table.
grep -E 'Fusarium|no-hit' ${prefix}*.table.txt > ${prefix}.FusariumHits.table.txt

echo "Generating list file..."
#Create a list of the Fusarium and no-hit only contigs. 
awk '{print $1}' ${prefix}.FusariumHits.table.txt >  ${prefix}.FusariumHits.list.txt

echo "Generating contaminant filtered fasta..."
#Create a list of the Fusarium and no-hit only contigs. 
blobtools seqfilter -i ${infile} -l ${prefix}.FusariumHits.list.txt -o ${prefix}.ContaminantFiltered

echo "Generating contaminant sequences fasta..."
#Create a list of the Fusarium and no-hit only contigs. 
blobtools seqfilter -v -i ${infile} -l ${prefix}.FusariumHits.list.txt -o ${prefix}.ContaminantSequences

echo "ContaminantFilter.sh finished."
echo $(date)
echo "+++++
It is advisable that you check the number of contigs in the ${prefix}.ContaminantFiltered.fasta matches the number of lines in the ${prefix}.FusariumHits.table.txt file."
python -c "print('=' * 75)" 

\end{lstlisting}

